tests/reference/tokens-test_literal-e20c024.stdout

```diff
@@ -0,0 +1,13 @@
+(KEYWORD "def") 0:2
```

Comment:
How did this file got commited? I think it should not be present.

Reply:
I have not committed this, as this is in stdout, so I believe it is due to updated references.

Reply:
>I have not committed this, as this is in stdout, so I believe it is due to updated references.

Can you trying doing:
```console
rm -rf tests/reference/* (then press 'y' if asked for confirmation)
./run_tests.py -u
git add tests
git commit -m "TEST: Update reference tests"
```

Reply:
I did this, here's the output:
```
(lp) kishan@kishan-IdeaPad-3-15IIL05:~/Desktop/lpython$ git commit -m "TEST: Update reference tests"
[replace 4bbd4cbdd] TEST: Update reference tests
 779 files changed, 107709 deletions(-)
 delete mode 100644 tests/reference/asr-array_01_decl-39cf894.json
 delete mode 100644 tests/reference/asr-array_01_decl-39cf894.stdout
 delete mode 100644 tests/reference/asr-array_02_decl-e8f6874.json
 delete mode 100644 tests/reference/asr-array_02_decl-e8f6874.stdout
 delete mode 100644 tests/reference/asr-arrays_01-a617b64.json
 delete mode 100644 tests/reference/asr-arrays_01-a617b64.stderr
 delete mode 100644 tests/reference/asr-arrays_02-da94458.json
 delete mode 100644 tests/reference/asr-arrays_02-da94458.stderr
 delete mode 100644 tests/reference/asr-arrays_03-de2e952.json
 delete mode 100644 tests/reference/asr-arrays_03-de2e952.stderr
 delete mode 100644 tests/reference/asr-arrays_04-880407c.json
 delete mode 100644 tests/reference/asr-arrays_04-880407c.stderr
 delete mode 100644 tests/reference/asr-arrays_05-ec8fbd5.json
 delete mode 100644 tests/reference/asr-arrays_05-ec8fbd5.stderr
 delete mode 100644 tests/reference/asr-arrays_06-fbb09a3.json
 delete mode 100644 tests/reference/asr-arrays_06-fbb09a3.stderr
 delete mode 100644 tests/reference/asr-arrays_07-de430fd.json
 delete mode 100644 tests/reference/asr-arrays_07-de430fd.stderr
 delete mode 100644 tests/reference/asr-arrays_08-ba317a3.json
 delete mode 100644 tests/reference/asr-arrays_08-ba317a3.stderr
 delete mode 100644 tests/reference/asr-arrays_09-50ee586.json
 delete mode 100644 tests/reference/asr-arrays_09-50ee586.stderr
 delete mode 100644 tests/reference/asr-arrays_10-bc82d75.json
 delete mode 100644 tests/reference/asr-arrays_10-bc82d75.stderr
 delete mode 100644 tests/reference/asr-arrays_11-fc505b4.json
 delete mode 100644 tests/reference/asr-arrays_11-fc505b4.stderr
 delete mode 100644 tests/reference/asr-arrays_12-63d6f25.json
 delete mode 100644 tests/reference/asr-arrays_12-63d6f25.stderr
 delete mode 100644 tests/reference/asr-arrays_13-b5fcc7e.json
 delete mode 100644 tests/reference/asr-arrays_13-b5fcc7e.stderr
 delete mode 100644 tests/reference/asr-arrays_14-78be00e.json
 delete mode 100644 tests/reference/asr-arrays_14-78be00e.stderr
 delete mode 100644 tests/reference/asr-assert1-1ce92ea.json
 delete mode 100644 tests/reference/asr-assert1-1ce92ea.stdout
 delete mode 100644 tests/reference/asr-assign1-886f049.json
 delete mode 100644 tests/reference/asr-assign1-886f049.stdout
 delete mode 100644 tests/reference/asr-assign2-8d1a2ee.json
 delete mode 100644 tests/reference/asr-assign2-8d1a2ee.stdout
 delete mode 100644 tests/reference/asr-bindc_01-6d521a9.json
 delete mode 100644 tests/reference/asr-bindc_01-6d521a9.stdout
 delete mode 100644 tests/reference/asr-bindc_01-f761165.json
 delete mode 100644 tests/reference/asr-bindc_01-f761165.stderr
 delete mode 100644 tests/reference/asr-bindc_02-5092d8e.json
 delete mode 100644 tests/reference/asr-bindc_02-5092d8e.stderr
 delete mode 100644 tests/reference/asr-bindc_02-bc1a7ea.json
 delete mode 100644 tests/reference/asr-bindc_02-bc1a7ea.stdout
 delete mode 100644 tests/reference/asr-bindc_03-95dbba7.json
 delete mode 100644 tests/reference/asr-bindc_03-95dbba7.stderr
 delete mode 100644 tests/reference/asr-bindc_04-06bd800.json
 delete mode 100644 tests/reference/asr-bindc_04-06bd800.stderr
 delete mode 100644 tests/reference/asr-bindc_10e-8b10394.json
 delete mode 100644 tests/reference/asr-bindc_10e-8b10394.stderr
 delete mode 100644 tests/reference/asr-c_interop1-cf2e9b4.json
 delete mode 100644 tests/reference/asr-c_interop1-cf2e9b4.stdout
 delete mode 100644 tests/reference/asr-callback_01-df40fd5.json
 delete mode 100644 tests/reference/asr-callback_01-df40fd5.stdout
 delete mode 100644 tests/reference/asr-cast-435c233.json
 delete mode 100644 tests/reference/asr-cast-435c233.stdout
 delete mode 100644 tests/reference/asr-complex1-f26c460.json
 delete mode 100644 tests/reference/asr-complex1-f26c460.stdout
 delete mode 100644 tests/reference/asr-const_01-af8289b.json
 delete mode 100644 tests/reference/asr-const_01-af8289b.stderr
 delete mode 100644 tests/reference/asr-const_02-fce29b7.json
 delete mode 100644 tests/reference/asr-const_02-fce29b7.stderr
 delete mode 100644 tests/reference/asr-constants1-5828e8a.json
 delete mode 100644 tests/reference/asr-constants1-5828e8a.stdout
 delete mode 100644 tests/reference/asr-cptr_01-4e660f1.json
 delete mode 100644 tests/reference/asr-cptr_01-4e660f1.stderr
 delete mode 100644 tests/reference/asr-dictionary1-a105a36.json
 delete mode 100644 tests/reference/asr-dictionary1-a105a36.stdout
 delete mode 100644 tests/reference/asr-doconcurrentloop_01-3fdc189.json
 delete mode 100644 tests/reference/asr-doconcurrentloop_01-3fdc189.stdout
 delete mode 100644 tests/reference/asr-elemental_01-b58df26.json
 delete mode 100644 tests/reference/asr-elemental_01-b58df26.stdout
 delete mode 100644 tests/reference/asr-enum_01-30e1b4a.json
 delete mode 100644 tests/reference/asr-enum_01-30e1b4a.stderr
 delete mode 100644 tests/reference/asr-enum_02-54656c5.json
 delete mode 100644 tests/reference/asr-enum_02-54656c5.stderr
 delete mode 100644 tests/reference/asr-expr1-8df2d66.json
 delete mode 100644 tests/reference/asr-expr1-8df2d66.stdout
 delete mode 100644 tests/reference/asr-expr10-efcbb1b.json
 delete mode 100644 tests/reference/asr-expr10-efcbb1b.stdout
 delete mode 100644 tests/reference/asr-expr11-9b91d35.json
 delete mode 100644 tests/reference/asr-expr11-9b91d35.stdout
 delete mode 100644 tests/reference/asr-expr12-5c5b71e.json
 delete mode 100644 tests/reference/asr-expr12-5c5b71e.stdout
 delete mode 100644 tests/reference/asr-expr13-81bdb5a.json
 delete mode 100644 tests/reference/asr-expr13-81bdb5a.stdout
 delete mode 100644 tests/reference/asr-expr16-a3dc453.json
 delete mode 100644 tests/reference/asr-expr16-a3dc453.stderr
 delete mode 100644 tests/reference/asr-expr2-2e78a12.json
 delete mode 100644 tests/reference/asr-expr2-2e78a12.stdout
 delete mode 100644 tests/reference/asr-expr4-cef6743.json
 delete mode 100644 tests/reference/asr-expr4-cef6743.stdout
 delete mode 100644 tests/reference/asr-expr5-645ffcc.json
 delete mode 100644 tests/reference/asr-expr5-645ffcc.stdout
 delete mode 100644 tests/reference/asr-expr6-368e5ed.json
 delete mode 100644 tests/reference/asr-expr6-368e5ed.stdout
 delete mode 100644 tests/reference/asr-expr7-480ba2f.json
 delete mode 100644 tests/reference/asr-expr7-480ba2f.stderr
 delete mode 100644 tests/reference/asr-expr7-480ba2f.stdout
 delete mode 100644 tests/reference/asr-expr8-6beda60.json
 delete mode 100644 tests/reference/asr-expr8-6beda60.stdout
 delete mode 100644 tests/reference/asr-expr9-814e4bc.json
 delete mode 100644 tests/reference/asr-expr9-814e4bc.stdout
 delete mode 100644 tests/reference/asr-expr_01-211000e.json
 delete mode 100644 tests/reference/asr-expr_01-211000e.stdout
 delete mode 100644 tests/reference/asr-expr_01-a0d4829.json
 delete mode 100644 tests/reference/asr-expr_01-a0d4829.stdout
 delete mode 100644 tests/reference/asr-expr_05-3a37324.json
 delete mode 100644 tests/reference/asr-expr_05-3a37324.stdout
 delete mode 100644 tests/reference/asr-expr_07-7742668.json
 delete mode 100644 tests/reference/asr-expr_07-7742668.stdout
 delete mode 100644 tests/reference/asr-expr_09-f3e89c8.json
 delete mode 100644 tests/reference/asr-expr_09-f3e89c8.stdout
 delete mode 100644 tests/reference/asr-expr_10-d39708c.json
 delete mode 100644 tests/reference/asr-expr_10-d39708c.stdout
 delete mode 100644 tests/reference/asr-expr_12-6769be0.json
 delete mode 100644 tests/reference/asr-expr_12-6769be0.stdout
 delete mode 100644 tests/reference/asr-expr_14-f2bd343.json
 delete mode 100644 tests/reference/asr-expr_14-f2bd343.stdout
 delete mode 100644 tests/reference/asr-func_01-d87aa4a.json
 delete mode 100644 tests/reference/asr-func_01-d87aa4a.stderr
 delete mode 100644 tests/reference/asr-func_02-b439474.json
 delete mode 100644 tests/reference/asr-func_02-b439474.stderr
 delete mode 100644 tests/reference/asr-func_03-cd744a0.json
 delete mode 100644 tests/reference/asr-func_03-cd744a0.stderr
 delete mode 100644 tests/reference/asr-func_04-eef2656.json
 delete mode 100644 tests/reference/asr-func_04-eef2656.stderr
 delete mode 100644 tests/reference/asr-func_05-c22b921.json
 delete mode 100644 tests/reference/asr-func_05-c22b921.stderr
 delete mode 100644 tests/reference/asr-func_06-62e738c.json
 delete mode 100644 tests/reference/asr-func_06-62e738c.stderr
 delete mode 100644 tests/reference/asr-func_07-4a8c076.json
 delete mode 100644 tests/reference/asr-func_07-4a8c076.stderr
 delete mode 100644 tests/reference/asr-func_08-c137415.json
 delete mode 100644 tests/reference/asr-func_08-c137415.stderr
 delete mode 100644 tests/reference/asr-func_inline_01-56af272.json
 delete mode 100644 tests/reference/asr-func_inline_01-56af272.stdout
 delete mode 100644 tests/reference/asr-generics_01-d616074.json
 delete mode 100644 tests/reference/asr-generics_01-d616074.stdout
 delete mode 100644 tests/reference/asr-generics_02-e2ea5c9.json
 delete mode 100644 tests/reference/asr-generics_02-e2ea5c9.stdout
 delete mode 100644 tests/reference/asr-generics_array_01-682b1b2.json
 delete mode 100644 tests/reference/asr-generics_array_01-682b1b2.stdout
 delete mode 100644 tests/reference/asr-generics_array_02-22c8dc1.json
 delete mode 100644 tests/reference/asr-generics_array_02-22c8dc1.stdout
 delete mode 100644 tests/reference/asr-generics_array_03-fb3706c.json
 delete mode 100644 tests/reference/asr-generics_array_03-fb3706c.stdout
 delete mode 100644 tests/reference/asr-generics_error_01-1e05cd6.json
 delete mode 100644 tests/reference/asr-generics_error_01-1e05cd6.stderr
 delete mode 100644 tests/reference/asr-generics_error_02-d614928.json
 delete mode 100644 tests/reference/asr-generics_error_02-d614928.stderr
 delete mode 100644 tests/reference/asr-generics_error_03-208d10d.json
 delete mode 100644 tests/reference/asr-generics_error_03-208d10d.stderr
 delete mode 100644 tests/reference/asr-generics_list_01-39c4044.json
 delete mode 100644 tests/reference/asr-generics_list_01-39c4044.stdout
 delete mode 100644 tests/reference/asr-global_scope1-354e217.json
 delete mode 100644 tests/reference/asr-global_scope1-354e217.stdout
 delete mode 100644 tests/reference/asr-global_syms_01-273906f.json
 delete mode 100644 tests/reference/asr-global_syms_01-273906f.stdout
 delete mode 100644 tests/reference/asr-intent_01-66824bc.json
 delete mode 100644 tests/reference/asr-intent_01-66824bc.stdout
 delete mode 100644 tests/reference/asr-kwargs_01_error-ab9530d.json
 delete mode 100644 tests/reference/asr-kwargs_01_error-ab9530d.stderr
 delete mode 100644 tests/reference/asr-kwargs_02_error-7c91f8f.json
 delete mode 100644 tests/reference/asr-kwargs_02_error-7c91f8f.stderr
 delete mode 100644 tests/reference/asr-kwargs_03_error-a68cc46.json
 delete mode 100644 tests/reference/asr-kwargs_03_error-a68cc46.stderr
 delete mode 100644 tests/reference/asr-lambda_01-1ec3e01.json
 delete mode 100644 tests/reference/asr-lambda_01-1ec3e01.stderr
 delete mode 100644 tests/reference/asr-list1-770ba33.json
 delete mode 100644 tests/reference/asr-list1-770ba33.stdout
 delete mode 100644 tests/reference/asr-loop1-10d3109.json
 delete mode 100644 tests/reference/asr-loop1-10d3109.stdout
 delete mode 100644 tests/reference/asr-loop3-a579196.json
 delete mode 100644 tests/reference/asr-loop3-a579196.stdout
 delete mode 100644 tests/reference/asr-loop4-3d3216e.json
 delete mode 100644 tests/reference/asr-loop4-3d3216e.stdout
 delete mode 100644 tests/reference/asr-loop_01-e7aa06a.json
 delete mode 100644 tests/reference/asr-loop_01-e7aa06a.stderr
 delete mode 100644 tests/reference/asr-loop_02-5741651.json
 delete mode 100644 tests/reference/asr-loop_02-5741651.stderr
 delete mode 100644 tests/reference/asr-loop_03-401ab45.json
 delete mode 100644 tests/reference/asr-loop_03-401ab45.stderr
 delete mode 100644 tests/reference/asr-modules_02-ec92e6f.json
 delete mode 100644 tests/reference/asr-modules_02-ec92e6f.stdout
 delete mode 100644 tests/reference/asr-print_02-afbe092.json
 delete mode 100644 tests/reference/asr-print_02-afbe092.stdout
 delete mode 100644 tests/reference/asr-print_list_tuple_03-9de3736.json
 delete mode 100644 tests/reference/asr-print_list_tuple_03-9de3736.stdout
 delete mode 100644 tests/reference/asr-set1-b7b913a.json
 delete mode 100644 tests/reference/asr-set1-b7b913a.stdout
 delete mode 100644 tests/reference/asr-string_01-78629c4.json
 delete mode 100644 tests/reference/asr-string_01-78629c4.stderr
 delete mode 100644 tests/reference/asr-string_02-499c9ff.json
 delete mode 100644 tests/reference/asr-string_02-499c9ff.stderr
 delete mode 100644 tests/reference/asr-structs_01-0893e35.json
 delete mode 100644 tests/reference/asr-structs_01-0893e35.stderr
 delete mode 100644 tests/reference/asr-structs_01-66dc2c9.json
 delete mode 100644 tests/reference/asr-structs_01-66dc2c9.stdout
 delete mode 100644 tests/reference/asr-structs_01-be14d49.json
 delete mode 100644 tests/reference/asr-structs_01-be14d49.stdout
 delete mode 100644 tests/reference/asr-structs_02-2ab459a.json
 delete mode 100644 tests/reference/asr-structs_02-2ab459a.stdout
 delete mode 100644 tests/reference/asr-structs_02-f95782c.json
 delete mode 100644 tests/reference/asr-structs_02-f95782c.stderr
 delete mode 100644 tests/reference/asr-structs_03-0cef911.json
 delete mode 100644 tests/reference/asr-structs_03-0cef911.stdout
 delete mode 100644 tests/reference/asr-structs_03-754fb64.json
 delete mode 100644 tests/reference/asr-structs_03-754fb64.stderr
 delete mode 100644 tests/reference/asr-structs_04-387747b.json
 delete mode 100644 tests/reference/asr-structs_04-387747b.stdout
 delete mode 100644 tests/reference/asr-structs_04-7b864bc.json
 delete mode 100644 tests/reference/asr-structs_04-7b864bc.stderr
 delete mode 100644 tests/reference/asr-structs_05-fa98307.json
 delete mode 100644 tests/reference/asr-structs_05-fa98307.stdout
 delete mode 100644 tests/reference/asr-structs_06-6e14537.json
 delete mode 100644 tests/reference/asr-structs_06-6e14537.stderr
 delete mode 100644 tests/reference/asr-structs_07-83694b7.json
 delete mode 100644 tests/reference/asr-structs_07-83694b7.stderr
 delete mode 100644 tests/reference/asr-structs_08-fa4dbf0.json
 delete mode 100644 tests/reference/asr-structs_08-fa4dbf0.stderr
 delete mode 100644 tests/reference/asr-structs_09-f3ffe08.json
 delete mode 100644 tests/reference/asr-structs_09-f3ffe08.stderr
 delete mode 100644 tests/reference/asr-structs_10-cb8a283.json
 delete mode 100644 tests/reference/asr-structs_10-cb8a283.stderr
 delete mode 100644 tests/reference/asr-structs_16-44de89a.json
 delete mode 100644 tests/reference/asr-structs_16-44de89a.stdout
 delete mode 100644 tests/reference/asr-subscript1-1acfc19.json
 delete mode 100644 tests/reference/asr-subscript1-1acfc19.stdout
 delete mode 100644 tests/reference/asr-test_annassign_01-2f18669.json
 delete mode 100644 tests/reference/asr-test_annassign_01-2f18669.stderr
 delete mode 100644 tests/reference/asr-test_annassign_02-accf6db.json
 delete mode 100644 tests/reference/asr-test_annassign_02-accf6db.stderr
 delete mode 100644 tests/reference/asr-test_annassign_03-f8c6160.json
 delete mode 100644 tests/reference/asr-test_annassign_03-f8c6160.stderr
 delete mode 100644 tests/reference/asr-test_annassign_type_mismatch-7dac7be.json
 delete mode 100644 tests/reference/asr-test_annassign_type_mismatch-7dac7be.stderr
 delete mode 100644 tests/reference/asr-test_annassign_type_mismatch2-fc883f7.json
 delete mode 100644 tests/reference/asr-test_annassign_type_mismatch2-fc883f7.stderr
 delete mode 100644 tests/reference/asr-test_append_type_mismatch-030e9c7.json
 delete mode 100644 tests/reference/asr-test_append_type_mismatch-030e9c7.stderr
 delete mode 100644 tests/reference/asr-test_assign1-a94d41e.json
 delete mode 100644 tests/reference/asr-test_assign1-a94d41e.stderr
 delete mode 100644 tests/reference/asr-test_assign2-fa29029.json
 delete mode 100644 tests/reference/asr-test_assign2-fa29029.stderr
 delete mode 100644 tests/reference/asr-test_assign3-cc6fc9d.json
 delete mode 100644 tests/reference/asr-test_assign3-cc6fc9d.stderr
 delete mode 100644 tests/reference/asr-test_assign4-a2136e0.json
 delete mode 100644 tests/reference/asr-test_assign4-a2136e0.stderr
 delete mode 100644 tests/reference/asr-test_assign5-694a98f.json
 delete mode 100644 tests/reference/asr-test_assign5-694a98f.stderr
 delete mode 100644 tests/reference/asr-test_assign6-05cd64f.json
 delete mode 100644 tests/reference/asr-test_assign6-05cd64f.stderr
 delete mode 100644 tests/reference/asr-test_assign7-beebac3.json
 delete mode 100644 tests/reference/asr-test_assign7-beebac3.stderr
 delete mode 100644 tests/reference/asr-test_assign8-4b26e63.json
 delete mode 100644 tests/reference/asr-test_assign8-4b26e63.stderr
 delete mode 100644 tests/reference/asr-test_assign9-51278b8.json
 delete mode 100644 tests/reference/asr-test_assign9-51278b8.stderr
 delete mode 100644 tests/reference/asr-test_async-361297c.json
 delete mode 100644 tests/reference/asr-test_async-361297c.stderr
 delete mode 100644 tests/reference/asr-test_binop1-50b63f6.json
 delete mode 100644 tests/reference/asr-test_binop1-50b63f6.stderr
 delete mode 100644 tests/reference/asr-test_binop2-e5749af.json
 delete mode 100644 tests/reference/asr-test_binop2-e5749af.stderr
 delete mode 100644 tests/reference/asr-test_binop3-a67201d.json
 delete mode 100644 tests/reference/asr-test_binop3-a67201d.stderr
 delete mode 100644 tests/reference/asr-test_bit_length-da3a264.json
 delete mode 100644 tests/reference/asr-test_bit_length-da3a264.stderr
 delete mode 100644 tests/reference/asr-test_bitwise_on_complex-dd9568b.json
 delete mode 100644 tests/reference/asr-test_bitwise_on_complex-dd9568b.stderr
 delete mode 100644 tests/reference/asr-test_bitwise_on_float-2e09b30.json
 delete mode 100644 tests/reference/asr-test_bitwise_on_float-2e09b30.stderr
 delete mode 100644 tests/reference/asr-test_bool_binop-f856ef0.json
 delete mode 100644 tests/reference/asr-test_bool_binop-f856ef0.stdout
 delete mode 100644 tests/reference/asr-test_builtin-aa64615.json
 delete mode 100644 tests/reference/asr-test_builtin-aa64615.stdout
 delete mode 100644 tests/reference/asr-test_builtin_abs-c74d2c9.json
 delete mode 100644 tests/reference/asr-test_builtin_abs-c74d2c9.stdout
 delete mode 100644 tests/reference/asr-test_builtin_bin-52ba9fa.json
 delete mode 100644 tests/reference/asr-test_builtin_bin-52ba9fa.stdout
 delete mode 100644 tests/reference/asr-test_builtin_bool-330223a.json
 delete mode 100644 tests/reference/asr-test_builtin_bool-330223a.stdout
 delete mode 100644 tests/reference/asr-test_builtin_float-20601dd.json
 delete mode 100644 tests/reference/asr-test_builtin_float-20601dd.stdout
 delete mode 100644 tests/reference/asr-test_builtin_hex-64bd268.json
 delete mode 100644 tests/reference/asr-test_builtin_hex-64bd268.stdout
 delete mode 100644 tests/reference/asr-test_builtin_int-8f88fdc.json
 delete mode 100644 tests/reference/asr-test_builtin_int-8f88fdc.stdout
 delete mode 100644 tests/reference/asr-test_builtin_len-55b0dec.json
 delete mode 100644 tests/reference/asr-test_builtin_len-55b0dec.stdout
 delete mode 100644 tests/reference/asr-test_builtin_oct-20b9066.json
 delete mode 100644 tests/reference/asr-test_builtin_oct-20b9066.stdout
 delete mode 100644 tests/reference/asr-test_builtin_pow-f02fcda.json
 delete mode 100644 tests/reference/asr-test_builtin_pow-f02fcda.stderr
 delete mode 100644 tests/reference/asr-test_builtin_pow-f02fcda.stdout
 delete mode 100644 tests/reference/asr-test_builtin_round-7417a21.json
 delete mode 100644 tests/reference/asr-test_builtin_round-7417a21.stdout
 delete mode 100644 tests/reference/asr-test_builtin_str-580e920.json
 delete mode 100644 tests/reference/asr-test_builtin_str-580e920.stdout
 delete mode 100644 tests/reference/asr-test_c_interop_01-e374f43.json
 delete mode 100644 tests/reference/asr-test_c_interop_01-e374f43.stdout
 delete mode 100644 tests/reference/asr-test_chr-91f4987.json
 delete mode 100644 tests/reference/asr-test_chr-91f4987.stderr
 delete mode 100644 tests/reference/asr-test_compare_01-c594bbe.json
 delete mode 100644 tests/reference/asr-test_compare_01-c594bbe.stderr
 delete mode 100644 tests/reference/asr-test_compare_02-bf2c76e.json
 delete mode 100644 tests/reference/asr-test_compare_02-bf2c76e.stderr
 delete mode 100644 tests/reference/asr-test_complex_01-a6def58.json
 delete mode 100644 tests/reference/asr-test_complex_01-a6def58.stdout
 delete mode 100644 tests/reference/asr-test_complex_02-782ba2d.json
 delete mode 100644 tests/reference/asr-test_complex_02-782ba2d.stdout
 delete mode 100644 tests/reference/asr-test_dict1-a45a075.json
 delete mode 100644 tests/reference/asr-test_dict1-a45a075.stderr
 delete mode 100644 tests/reference/asr-test_dict10-8c0beff.json
 delete mode 100644 tests/reference/asr-test_dict10-8c0beff.stderr
 delete mode 100644 tests/reference/asr-test_dict11-2ab4e6c.json
 delete mode 100644 tests/reference/asr-test_dict11-2ab4e6c.stderr
 delete mode 100644 tests/reference/asr-test_dict12-7c74d08.json
 delete mode 100644 tests/reference/asr-test_dict12-7c74d08.stderr
 delete mode 100644 tests/reference/asr-test_dict13-683b290.json
 delete mode 100644 tests/reference/asr-test_dict13-683b290.stderr
 delete mode 100644 tests/reference/asr-test_dict2-4587f02.json
 delete mode 100644 tests/reference/asr-test_dict2-4587f02.stderr
 delete mode 100644 tests/reference/asr-test_dict3-d28f38f.json
 delete mode 100644 tests/reference/asr-test_dict3-d28f38f.stderr
 delete mode 100644 tests/reference/asr-test_dict4-39489fa.json
 delete mode 100644 tests/reference/asr-test_dict4-39489fa.stderr
 delete mode 100644 tests/reference/asr-test_dict5-c436d37.json
 delete mode 100644 tests/reference/asr-test_dict5-c436d37.stderr
 delete mode 100644 tests/reference/asr-test_dict6-95e3b14.json
 delete mode 100644 tests/reference/asr-test_dict6-95e3b14.stderr
 delete mode 100644 tests/reference/asr-test_dict7-1415e14.json
 delete mode 100644 tests/reference/asr-test_dict7-1415e14.stderr
 delete mode 100644 tests/reference/asr-test_dict8-d960ce0.json
 delete mode 100644 tests/reference/asr-test_dict8-d960ce0.stderr
 delete mode 100644 tests/reference/asr-test_dict9-907bda7.json
 delete mode 100644 tests/reference/asr-test_dict9-907bda7.stderr
 delete mode 100644 tests/reference/asr-test_end_sep_keywords-2226a67.json
 delete mode 100644 tests/reference/asr-test_end_sep_keywords-2226a67.stdout
 delete mode 100644 tests/reference/asr-test_float_semantic_error-58c0c90.json
 delete mode 100644 tests/reference/asr-test_float_semantic_error-58c0c90.stderr
 delete mode 100644 tests/reference/asr-test_for2-e5d893f.json
 delete mode 100644 tests/reference/asr-test_for2-e5d893f.stderr
 delete mode 100644 tests/reference/asr-test_func_args-a898a72.json
 delete mode 100644 tests/reference/asr-test_func_args-a898a72.stderr
 delete mode 100644 tests/reference/asr-test_import_01-b859c43.json
 delete mode 100644 tests/reference/asr-test_import_01-b859c43.stderr
 delete mode 100644 tests/reference/asr-test_import_02-55b47fa.json
 delete mode 100644 tests/reference/asr-test_import_02-55b47fa.stderr
 delete mode 100644 tests/reference/asr-test_int_semantic_error-44fe25e.json
 delete mode 100644 tests/reference/asr-test_int_semantic_error-44fe25e.stderr
 delete mode 100644 tests/reference/asr-test_len1-839a615.json
 delete mode 100644 tests/reference/asr-test_len1-839a615.stderr
 delete mode 100644 tests/reference/asr-test_list1-73fd538.json
 delete mode 100644 tests/reference/asr-test_list1-73fd538.stderr
 delete mode 100644 tests/reference/asr-test_list2-10ffdd7.json
 delete mode 100644 tests/reference/asr-test_list2-10ffdd7.stderr
 delete mode 100644 tests/reference/asr-test_list3-5f4d2a8.json
 delete mode 100644 tests/reference/asr-test_list3-5f4d2a8.stderr
 delete mode 100644 tests/reference/asr-test_list4-d18a7e3.json
 delete mode 100644 tests/reference/asr-test_list4-d18a7e3.stderr
 delete mode 100644 tests/reference/asr-test_list_concat-41d186f.json
 delete mode 100644 tests/reference/asr-test_list_concat-41d186f.stderr
 delete mode 100644 tests/reference/asr-test_list_count-4b42498.json
 delete mode 100644 tests/reference/asr-test_list_count-4b42498.stderr
 delete mode 100644 tests/reference/asr-test_list_slicing-984fbf0.json
 delete mode 100644 tests/reference/asr-test_list_slicing-984fbf0.stderr
 delete mode 100644 tests/reference/asr-test_max_min-3c2fc51.json
 delete mode 100644 tests/reference/asr-test_max_min-3c2fc51.stdout
 delete mode 100644 tests/reference/asr-test_numpy_03-e600a49.json
 delete mode 100644 tests/reference/asr-test_numpy_03-e600a49.stdout
 delete mode 100644 tests/reference/asr-test_numpy_04-ecbb614.json
 delete mode 100644 tests/reference/asr-test_numpy_04-ecbb614.stdout
 delete mode 100644 tests/reference/asr-test_operator_01-0f232bf.json
 delete mode 100644 tests/reference/asr-test_operator_01-0f232bf.stderr
 delete mode 100644 tests/reference/asr-test_ord-316ce61.json
 delete mode 100644 tests/reference/asr-test_ord-316ce61.stderr
 delete mode 100644 tests/reference/asr-test_pointer_types-1bf0d01.json
 delete mode 100644 tests/reference/asr-test_pointer_types-1bf0d01.stderr
 delete mode 100644 tests/reference/asr-test_pow-3f5d550.json
 delete mode 100644 tests/reference/asr-test_pow-3f5d550.stderr
 delete mode 100644 tests/reference/asr-test_pow-3f5d550.stdout
 delete mode 100644 tests/reference/asr-test_pow1-581ef42.json
 delete mode 100644 tests/reference/asr-test_pow1-581ef42.stderr
 delete mode 100644 tests/reference/asr-test_pow2-0dcbd7d.json
 delete mode 100644 tests/reference/asr-test_pow2-0dcbd7d.stderr
 delete mode 100644 tests/reference/asr-test_pow3-79749ed.json
 delete mode 100644 tests/reference/asr-test_pow3-79749ed.stderr
 delete mode 100644 tests/reference/asr-test_pow4-ef60978.json
 delete mode 100644 tests/reference/asr-test_pow4-ef60978.stderr
 delete mode 100644 tests/reference/asr-test_print1-f1f36f1.json
 delete mode 100644 tests/reference/asr-test_print1-f1f36f1.stderr
 delete mode 100644 tests/reference/asr-test_print2-64acb15.json
 delete mode 100644 tests/reference/asr-test_print2-64acb15.stderr
 delete mode 100644 tests/reference/asr-test_return1-2603c9e.json
 delete mode 100644 tests/reference/asr-test_return1-2603c9e.stderr
 delete mode 100644 tests/reference/asr-test_return2-add829b.json
 delete mode 100644 tests/reference/asr-test_return2-add829b.stderr
 delete mode 100644 tests/reference/asr-test_set1-11379c7.json
 delete mode 100644 tests/reference/asr-test_set1-11379c7.stderr
 delete mode 100644 tests/reference/asr-test_set2-d91a6f0.json
 delete mode 100644 tests/reference/asr-test_set2-d91a6f0.stderr
 delete mode 100644 tests/reference/asr-test_set3-f9bbffb.json
 delete mode 100644 tests/reference/asr-test_set3-f9bbffb.stderr
 delete mode 100644 tests/reference/asr-test_set4-53fea39.json
 delete mode 100644 tests/reference/asr-test_set4-53fea39.stderr
 delete mode 100644 tests/reference/asr-test_set_indexing-a1c21b8.json
 delete mode 100644 tests/reference/asr-test_set_indexing-a1c21b8.stderr
 delete mode 100644 tests/reference/asr-test_str_capitalize-9f8dfa9.json
 delete mode 100644 tests/reference/asr-test_str_capitalize-9f8dfa9.stderr
 delete mode 100644 tests/reference/asr-test_str_indexing-b200a4e.json
 delete mode 100644 tests/reference/asr-test_str_indexing-b200a4e.stderr
 delete mode 100644 tests/reference/asr-test_str_slicing-eca1381.json
 delete mode 100644 tests/reference/asr-test_str_slicing-eca1381.stderr
 delete mode 100644 tests/reference/asr-test_str_slicing2-2f07e9a.json
 delete mode 100644 tests/reference/asr-test_str_slicing2-2f07e9a.stderr
 delete mode 100644 tests/reference/asr-test_str_slicing3-fe6a03c.json
 delete mode 100644 tests/reference/asr-test_str_slicing3-fe6a03c.stderr
 delete mode 100644 tests/reference/asr-test_str_slicing4-a0c7a69.json
 delete mode 100644 tests/reference/asr-test_str_slicing4-a0c7a69.stderr
 delete mode 100644 tests/reference/asr-test_str_to_int-61553e7.json
 delete mode 100644 tests/reference/asr-test_str_to_int-61553e7.stderr
 delete mode 100644 tests/reference/asr-test_tuple1-7abe88f.json
 delete mode 100644 tests/reference/asr-test_tuple1-7abe88f.stderr
 delete mode 100644 tests/reference/asr-test_tuple2-b046610.json
 delete mode 100644 tests/reference/asr-test_tuple2-b046610.stderr
 delete mode 100644 tests/reference/asr-test_tuple3-19fa47f.json
 delete mode 100644 tests/reference/asr-test_tuple3-19fa47f.stderr
 delete mode 100644 tests/reference/asr-test_type_mismatch_01-09e8af3.json
 delete mode 100644 tests/reference/asr-test_type_mismatch_01-09e8af3.stderr
 delete mode 100644 tests/reference/asr-test_unary_op_03-e799eae.json
 delete mode 100644 tests/reference/asr-test_unary_op_03-e799eae.stdout
 delete mode 100644 tests/reference/asr-test_unsupported_type-0d813dd.json
 delete mode 100644 tests/reference/asr-test_unsupported_type-0d813dd.stderr
 delete mode 100644 tests/reference/asr-test_zero_division-3dd84e8.json
 delete mode 100644 tests/reference/asr-test_zero_division-3dd84e8.stderr
 delete mode 100644 tests/reference/asr-test_zero_division2-d84989f.json
 delete mode 100644 tests/reference/asr-test_zero_division2-d84989f.stderr
 delete mode 100644 tests/reference/asr-test_zero_division3-29efb9e.json
 delete mode 100644 tests/reference/asr-test_zero_division3-29efb9e.stderr
 delete mode 100644 tests/reference/asr-test_zero_division4-bf4af64.json
 delete mode 100644 tests/reference/asr-test_zero_division4-bf4af64.stderr
 delete mode 100644 tests/reference/asr-tuple1-09972ab.json
 delete mode 100644 tests/reference/asr-tuple1-09972ab.stdout
 delete mode 100644 tests/reference/asr-type_mismatch-61052c7.json
 delete mode 100644 tests/reference/asr-type_mismatch-61052c7.stderr
 delete mode 100644 tests/reference/asr-unsigned_01-892b178.json
 delete mode 100644 tests/reference/asr-unsigned_01-892b178.stderr
 delete mode 100644 tests/reference/asr-unsigned_02-6563e58.json
 delete mode 100644 tests/reference/asr-unsigned_02-6563e58.stderr
 delete mode 100644 tests/reference/asr-unsigned_03-f785652.json
 delete mode 100644 tests/reference/asr-unsigned_03-f785652.stderr
 delete mode 100644 tests/reference/asr-unsigned_04-c856d3a.json
 delete mode 100644 tests/reference/asr-unsigned_04-c856d3a.stderr
 delete mode 100644 tests/reference/asr-vec_01-66ac423.json
 delete mode 100644 tests/reference/asr-vec_01-66ac423.stdout
 delete mode 100644 tests/reference/asr_json-modules_02-53952e6.json
 delete mode 100644 tests/reference/asr_json-modules_02-53952e6.stdout
 delete mode 100644 tests/reference/ast-assert1-b0154ee.json
 delete mode 100644 tests/reference/ast-assert1-b0154ee.stdout
 delete mode 100644 tests/reference/ast-assign1-2a4c9ed.json
 delete mode 100644 tests/reference/ast-assign1-2a4c9ed.stdout
 delete mode 100644 tests/reference/ast-complex1-800b4bb.json
 delete mode 100644 tests/reference/ast-complex1-800b4bb.stdout
 delete mode 100644 tests/reference/ast-constants1-91cb6ff.json
 delete mode 100644 tests/reference/ast-constants1-91cb6ff.stdout
 delete mode 100644 tests/reference/ast-dictionary1-1a7e00a.json
 delete mode 100644 tests/reference/ast-dictionary1-1a7e00a.stdout
 delete mode 100644 tests/reference/ast-doconcurrentloop_01-ed7017b.json
 delete mode 100644 tests/reference/ast-doconcurrentloop_01-ed7017b.stdout
 delete mode 100644 tests/reference/ast-ellipsis1-4f6c4dd.json
 delete mode 100644 tests/reference/ast-ellipsis1-4f6c4dd.stdout
 delete mode 100644 tests/reference/ast-expr1-1e8f7b1.json
 delete mode 100644 tests/reference/ast-expr1-1e8f7b1.stdout
 delete mode 100644 tests/reference/ast-expr10-a8d646d.json
 delete mode 100644 tests/reference/ast-expr10-a8d646d.stdout
 delete mode 100644 tests/reference/ast-expr11-1d29f78.json
 delete mode 100644 tests/reference/ast-expr11-1d29f78.stdout
 delete mode 100644 tests/reference/ast-expr12-adaecda.json
 delete mode 100644 tests/reference/ast-expr12-adaecda.stdout
 delete mode 100644 tests/reference/ast-expr13-c35ace1.json
 delete mode 100644 tests/reference/ast-expr13-c35ace1.stdout
 delete mode 100644 tests/reference/ast-expr2-6642d4a.json
 delete mode 100644 tests/reference/ast-expr2-6642d4a.stdout
 delete mode 100644 tests/reference/ast-expr4-49316cb.json
 delete mode 100644 tests/reference/ast-expr4-49316cb.stdout
 delete mode 100644 tests/reference/ast-expr5-bbc6e71.json
 delete mode 100644 tests/reference/ast-expr5-bbc6e71.stdout
 delete mode 100644 tests/reference/ast-expr6-0b12a67.json
 delete mode 100644 tests/reference/ast-expr6-0b12a67.stdout
 delete mode 100644 tests/reference/ast-expr7-fe52776.json
 delete mode 100644 tests/reference/ast-expr7-fe52776.stdout
 delete mode 100644 tests/reference/ast-expr8-7db6b28.json
 delete mode 100644 tests/reference/ast-expr8-7db6b28.stdout
 delete mode 100644 tests/reference/ast-expr9-d184496.json
 delete mode 100644 tests/reference/ast-expr9-d184496.stdout
 delete mode 100644 tests/reference/ast-expr_01-d0927f9.json
 delete mode 100644 tests/reference/ast-expr_01-d0927f9.stdout
 delete mode 100644 tests/reference/ast-global1-b2690cf.json
 delete mode 100644 tests/reference/ast-global1-b2690cf.stdout
 delete mode 100644 tests/reference/ast-global_scope1-1d68a6c.json
 delete mode 100644 tests/reference/ast-global_scope1-1d68a6c.stdout
 delete mode 100644 tests/reference/ast-list1-9ce2da0.json
 delete mode 100644 tests/reference/ast-list1-9ce2da0.stdout
 delete mode 100644 tests/reference/ast-loop1-194a137.json
 delete mode 100644 tests/reference/ast-loop1-194a137.stdout
 delete mode 100644 tests/reference/ast-loop3-f7e0393.json
 delete mode 100644 tests/reference/ast-loop3-f7e0393.stdout
 delete mode 100644 tests/reference/ast-prefix_string_01-cf221fd.json
 delete mode 100644 tests/reference/ast-prefix_string_01-cf221fd.stderr
 delete mode 100644 tests/reference/ast-prefix_string_02-3d530b2.json
 delete mode 100644 tests/reference/ast-prefix_string_02-3d530b2.stderr
 delete mode 100644 tests/reference/ast-set1-ebd6ee0.json
 delete mode 100644 tests/reference/ast-set1-ebd6ee0.stdout
 delete mode 100644 tests/reference/ast-subscript1-bd5584b.json
 delete mode 100644 tests/reference/ast-subscript1-bd5584b.stdout
 delete mode 100644 tests/reference/ast-tuple1-2fb5396.json
 delete mode 100644 tests/reference/ast-tuple1-2fb5396.stdout
 delete mode 100644 tests/reference/ast_new-async1-b3d07ed.json
 delete mode 100644 tests/reference/ast_new-async1-b3d07ed.stdout
 delete mode 100644 tests/reference/ast_new-boolOp1-478328f.json
 delete mode 100644 tests/reference/ast_new-boolOp1-478328f.stdout
 delete mode 100644 tests/reference/ast_new-class_def1-fe69291.json
 delete mode 100644 tests/reference/ast_new-class_def1-fe69291.stdout
 delete mode 100644 tests/reference/ast_new-class_def2-c6db986.json
 delete mode 100644 tests/reference/ast_new-class_def2-c6db986.stdout
 delete mode 100644 tests/reference/ast_new-comment2-f0984d5.json
 delete mode 100644 tests/reference/ast_new-comment2-f0984d5.stdout
 delete mode 100644 tests/reference/ast_new-comprehension1-69cf2af.json
 delete mode 100644 tests/reference/ast_new-comprehension1-69cf2af.stdout
 delete mode 100644 tests/reference/ast_new-conditional_expr1-07ccb9e.json
 delete mode 100644 tests/reference/ast_new-conditional_expr1-07ccb9e.stdout
 delete mode 100644 tests/reference/ast_new-dictionary1-445e718.json
 delete mode 100644 tests/reference/ast_new-dictionary1-445e718.stdout
 delete mode 100644 tests/reference/ast_new-ellipsis2-3a9750b.json
 delete mode 100644 tests/reference/ast_new-ellipsis2-3a9750b.stdout
 delete mode 100644 tests/reference/ast_new-for1-887432e.json
 delete mode 100644 tests/reference/ast_new-for1-887432e.stdout
 delete mode 100644 tests/reference/ast_new-for2-af08901.json
 delete mode 100644 tests/reference/ast_new-for2-af08901.stdout
 delete mode 100644 tests/reference/ast_new-function_def1-1a872df.json
 delete mode 100644 tests/reference/ast_new-function_def1-1a872df.stdout
 delete mode 100644 tests/reference/ast_new-function_def2-52c4587.json
 delete mode 100644 tests/reference/ast_new-function_def2-52c4587.stdout
 delete mode 100644 tests/reference/ast_new-function_def3-f66064a.json
 delete mode 100644 tests/reference/ast_new-function_def3-f66064a.stdout
 delete mode 100644 tests/reference/ast_new-global1-38edfbd.json
 delete mode 100644 tests/reference/ast_new-global1-38edfbd.stdout
 delete mode 100644 tests/reference/ast_new-if1-db43586.json
 delete mode 100644 tests/reference/ast_new-if1-db43586.stdout
 delete mode 100644 tests/reference/ast_new-if2-c3b6022.json
 delete mode 100644 tests/reference/ast_new-if2-c3b6022.stdout
 delete mode 100644 tests/reference/ast_new-import1-f643fd3.json
 delete mode 100644 tests/reference/ast_new-import1-f643fd3.stdout
 delete mode 100644 tests/reference/ast_new-lambda1-260d046.json
 delete mode 100644 tests/reference/ast_new-lambda1-260d046.stdout
 delete mode 100644 tests/reference/ast_new-lambda2-d84336e.json
 delete mode 100644 tests/reference/ast_new-lambda2-d84336e.stdout
 delete mode 100644 tests/reference/ast_new-match_stmt1-9e84d24.json
 delete mode 100644 tests/reference/ast_new-match_stmt1-9e84d24.stdout
 delete mode 100644 tests/reference/ast_new-slice1-9c440e3.json
 delete mode 100644 tests/reference/ast_new-slice1-9c440e3.stdout
 delete mode 100644 tests/reference/ast_new-statements1-e081093.json
 delete mode 100644 tests/reference/ast_new-statements1-e081093.stdout
 delete mode 100644 tests/reference/ast_new-statements2-c4cdc5f.json
 delete mode 100644 tests/reference/ast_new-statements2-c4cdc5f.stdout
 delete mode 100644 tests/reference/ast_new-string1-96b90b3.json
 delete mode 100644 tests/reference/ast_new-string1-96b90b3.stdout
 delete mode 100644 tests/reference/ast_new-string2-44323ea.json
 delete mode 100644 tests/reference/ast_new-string2-44323ea.stdout
 delete mode 100644 tests/reference/ast_new-string3-37f35a0.json
 delete mode 100644 tests/reference/ast_new-string3-37f35a0.stdout
 delete mode 100644 tests/reference/ast_new-try1-a9a22cf.json
 delete mode 100644 tests/reference/ast_new-try1-a9a22cf.stdout
 delete mode 100644 tests/reference/ast_new-tuple1-29c08af.json
 delete mode 100644 tests/reference/ast_new-tuple1-29c08af.stdout
 delete mode 100644 tests/reference/ast_new-type_comment1-710ea6c.json
 delete mode 100644 tests/reference/ast_new-type_comment1-710ea6c.stdout
 delete mode 100644 tests/reference/ast_new-unicode-d3199dc.json
 delete mode 100644 tests/reference/ast_new-unicode-d3199dc.stdout
 delete mode 100644 tests/reference/ast_new-while1-a4c6382.json
 delete mode 100644 tests/reference/ast_new-while1-a4c6382.stdout
 delete mode 100644 tests/reference/ast_new-with1-6c88c0f.json
 delete mode 100644 tests/reference/ast_new-with1-6c88c0f.stdout
 delete mode 100644 tests/reference/ast_new-yield-4c41668.json
 delete mode 100644 tests/reference/ast_new-yield-4c41668.stdout
 delete mode 100644 tests/reference/c-c_interop1-e215531.json
 delete mode 100644 tests/reference/c-c_interop1-e215531.stdout
 delete mode 100644 tests/reference/c-expr7-bb2692a.json
 delete mode 100644 tests/reference/c-expr7-bb2692a.stderr
 delete mode 100644 tests/reference/c-expr7-bb2692a.stdout
 delete mode 100644 tests/reference/c-expr_01-28f449f.json
 delete mode 100644 tests/reference/c-expr_01-28f449f.stdout
 delete mode 100644 tests/reference/c-expr_11-c452314.json
 delete mode 100644 tests/reference/c-expr_11-c452314.stdout
 delete mode 100644 tests/reference/c-expr_12-93c7780.json
 delete mode 100644 tests/reference/c-expr_12-93c7780.stdout
 delete mode 100644 tests/reference/c-func_static_01-fc146ec.json
 delete mode 100644 tests/reference/c-func_static_01-fc146ec.stdout
 delete mode 100644 tests/reference/c-import_order_01-3ebf3c3.json
 delete mode 100644 tests/reference/c-import_order_01-3ebf3c3.stdout
 delete mode 100644 tests/reference/c-loop1-3e341c7.json
 delete mode 100644 tests/reference/c-loop1-3e341c7.stdout
 delete mode 100644 tests/reference/c-loop4-eec10d3.json
 delete mode 100644 tests/reference/c-loop4-eec10d3.stdout
 delete mode 100644 tests/reference/c-print_01-4d44628.json
 delete mode 100644 tests/reference/c-print_01-4d44628.stdout
 delete mode 100644 tests/reference/c-test_import_02-d2c54c4.json
 delete mode 100644 tests/reference/c-test_import_02-d2c54c4.stdout
 delete mode 100644 tests/reference/c-test_issue_518-fbbd299.json
 delete mode 100644 tests/reference/c-test_issue_518-fbbd299.stdout
 delete mode 100644 tests/reference/c-variable_decl_03-fa1823b.json
 delete mode 100644 tests/reference/c-variable_decl_03-fa1823b.stdout
 delete mode 100644 tests/reference/cpp-assert1-ba60925.json
 delete mode 100644 tests/reference/cpp-assert1-ba60925.stdout
 delete mode 100644 tests/reference/cpp-doconcurrentloop_01-4e9f274.json
 delete mode 100644 tests/reference/cpp-doconcurrentloop_01-4e9f274.stdout
 delete mode 100644 tests/reference/cpp-expr12-fd2ea87.json
 delete mode 100644 tests/reference/cpp-expr12-fd2ea87.stdout
 delete mode 100644 tests/reference/cpp-expr15-1661c0d.json
 delete mode 100644 tests/reference/cpp-expr15-1661c0d.stdout
 delete mode 100644 tests/reference/cpp-expr2-09c05ad.json
 delete mode 100644 tests/reference/cpp-expr2-09c05ad.stdout
 delete mode 100644 tests/reference/cpp-expr5-1de0e30.json
 delete mode 100644 tests/reference/cpp-expr5-1de0e30.stdout
 delete mode 100644 tests/reference/cpp-expr6-f337f4f.json
 delete mode 100644 tests/reference/cpp-expr6-f337f4f.stdout
 delete mode 100644 tests/reference/cpp-expr7-529bd53.json
 delete mode 100644 tests/reference/cpp-expr7-529bd53.stderr
 delete mode 100644 tests/reference/cpp-expr7-529bd53.stdout
 delete mode 100644 tests/reference/cpp-expr8-704cece.json
 delete mode 100644 tests/reference/cpp-expr8-704cece.stdout
 delete mode 100644 tests/reference/cpp-expr9-48868e9.json
 delete mode 100644 tests/reference/cpp-expr9-48868e9.stdout
 delete mode 100644 tests/reference/cpp-expr_11-422c839.json
 delete mode 100644 tests/reference/cpp-expr_11-422c839.stdout
 delete mode 100644 tests/reference/cpp-loop1-0a8cf3b.json
 delete mode 100644 tests/reference/cpp-loop1-0a8cf3b.stdout
 delete mode 100644 tests/reference/cpp-loop3-6020091.json
 delete mode 100644 tests/reference/cpp-loop3-6020091.stdout
 delete mode 100644 tests/reference/cpp-loop4-cdb2174.json
 delete mode 100644 tests/reference/cpp-loop4-cdb2174.stdout
 delete mode 100644 tests/reference/cpp-print_01-026ef17.json
 delete mode 100644 tests/reference/cpp-print_01-026ef17.stdout
 delete mode 100644 tests/reference/cpp-test_builtin_pow-56b3f92.json
 delete mode 100644 tests/reference/cpp-test_builtin_pow-56b3f92.stderr
 delete mode 100644 tests/reference/cpp-test_builtin_pow-56b3f92.stdout
 delete mode 100644 tests/reference/cpp-test_list_repeat2-698d7f4.json
 delete mode 100644 tests/reference/cpp-test_list_repeat2-698d7f4.stdout
 delete mode 100644 tests/reference/cpp-test_unary_op_03-fd9669a.json
 delete mode 100644 tests/reference/cpp-test_unary_op_03-fd9669a.stdout
 delete mode 100644 tests/reference/llvm-assert1-8df4f31.json
 delete mode 100644 tests/reference/llvm-assert1-8df4f31.stdout
 delete mode 100644 tests/reference/llvm-bindc_01-c984f09.json
 delete mode 100644 tests/reference/llvm-bindc_01-c984f09.stdout
 delete mode 100644 tests/reference/llvm-bool1-af4376b.json
 delete mode 100644 tests/reference/llvm-bool1-af4376b.stdout
 delete mode 100644 tests/reference/llvm-expr14-b96b5b1.json
 delete mode 100644 tests/reference/llvm-expr14-b96b5b1.stdout
 delete mode 100644 tests/reference/llvm-expr_01-54467c1.json
 delete mode 100644 tests/reference/llvm-expr_01-54467c1.stdout
 delete mode 100644 tests/reference/llvm-func_inline_01-2d4583a.json
 delete mode 100644 tests/reference/llvm-func_inline_01-2d4583a.stdout
 delete mode 100644 tests/reference/llvm-lpython1-23c5987.json
 delete mode 100644 tests/reference/llvm-lpython1-23c5987.stdout
 delete mode 100644 tests/reference/llvm-print_04-443a8d8.json
 delete mode 100644 tests/reference/llvm-print_04-443a8d8.stdout
 delete mode 100644 tests/reference/llvm-structs_11-09fea6a.json
 delete mode 100644 tests/reference/llvm-structs_11-09fea6a.stdout
 delete mode 100644 tests/reference/llvm-test_issue_518-cdb641a.json
 delete mode 100644 tests/reference/llvm-test_issue_518-cdb641a.stdout
 delete mode 100644 tests/reference/llvm-test_unary_op_03-046fb86.json
 delete mode 100644 tests/reference/llvm-test_unary_op_03-046fb86.stdout
 delete mode 100644 tests/reference/llvm_dbg-expr_01-9fc5f30.json
 delete mode 100644 tests/reference/llvm_dbg-expr_01-9fc5f30.stdout
 delete mode 100644 tests/reference/pass_class_constructor-structs_16-5e3508f.json
 delete mode 100644 tests/reference/pass_class_constructor-structs_16-5e3508f.stdout
 delete mode 100644 tests/reference/pass_inline_function_calls-func_inline_01-8b6a5da.json
 delete mode 100644 tests/reference/pass_inline_function_calls-func_inline_01-8b6a5da.stdout
 delete mode 100644 tests/reference/pass_loop_vectorise-vec_01-be9985e.json
 delete mode 100644 tests/reference/pass_loop_vectorise-vec_01-be9985e.stdout
 delete mode 100644 tests/reference/pass_print_list_tuple-print_02-09600eb.json
 delete mode 100644 tests/reference/pass_print_list_tuple-print_02-09600eb.stdout
 delete mode 100644 tests/reference/pass_print_list_tuple-print_list_tuple_03-195fa9c.json
 delete mode 100644 tests/reference/pass_print_list_tuple-print_list_tuple_03-195fa9c.stdout
 delete mode 100644 tests/reference/python-assert1-192ca6c.json
 delete mode 100644 tests/reference/python-assert1-192ca6c.stdout
 delete mode 100644 tests/reference/python-assign1-f87bafa.json
 delete mode 100644 tests/reference/python-assign1-f87bafa.stdout
 delete mode 100644 tests/reference/python-expr11-e6681c8.json
 delete mode 100644 tests/reference/python-expr11-e6681c8.stdout
 delete mode 100644 tests/reference/python-expr17-3b84714.json
 delete mode 100644 tests/reference/python-expr17-3b84714.stdout
 delete mode 100644 tests/reference/python-expr2-6b69018.json
 delete mode 100644 tests/reference/python-expr2-6b69018.stdout
 delete mode 100644 tests/reference/python-expr4-161a0ec.json
 delete mode 100644 tests/reference/python-expr4-161a0ec.stdout
 delete mode 100644 tests/reference/python-expr5-dee0e5c.json
 delete mode 100644 tests/reference/python-expr5-dee0e5c.stdout
 delete mode 100644 tests/reference/python-expr6-1a1d4fb.json
 delete mode 100644 tests/reference/python-expr6-1a1d4fb.stdout
 delete mode 100644 tests/reference/run_dbg-test_assert_01-2f34744.json
 delete mode 100644 tests/reference/run_dbg-test_assert_01-2f34744.stderr
 delete mode 100644 tests/reference/run_dbg-test_assert_02-c6de25a.json
 delete mode 100644 tests/reference/run_dbg-test_assert_02-c6de25a.stderr
 delete mode 100644 tests/reference/run_dbg-test_assert_03-bd7b7dd.json
 delete mode 100644 tests/reference/run_dbg-test_assert_03-bd7b7dd.stderr
 delete mode 100644 tests/reference/run_dbg-test_quit_01-30889cc.json
 delete mode 100644 tests/reference/run_dbg-test_quit_01-30889cc.stderr
 delete mode 100644 tests/reference/run_dbg-test_raise_01-dfd86ca.json
 delete mode 100644 tests/reference/run_dbg-test_raise_01-dfd86ca.stderr
 delete mode 100644 tests/reference/runtime-test_dict14-421fe53.json
 delete mode 100644 tests/reference/runtime-test_dict14-421fe53.stderr
 delete mode 100644 tests/reference/runtime-test_dict15-6f3af0d.json
 delete mode 100644 tests/reference/runtime-test_dict15-6f3af0d.stderr
 delete mode 100644 tests/reference/runtime-test_dict16-c5a958d.json
 delete mode 100644 tests/reference/runtime-test_dict16-c5a958d.stderr
 delete mode 100644 tests/reference/runtime-test_list_01-3ee9b3e.json
 delete mode 100644 tests/reference/runtime-test_list_01-3ee9b3e.stderr
 delete mode 100644 tests/reference/runtime-test_list_02-5f7db5f.json
 delete mode 100644 tests/reference/runtime-test_list_02-5f7db5f.stderr
 delete mode 100644 tests/reference/runtime-test_list_index-0483808.json
 delete mode 100644 tests/reference/runtime-test_list_index-0483808.stderr
 delete mode 100644 tests/reference/runtime-test_list_index2-1249233.json
 delete mode 100644 tests/reference/runtime-test_list_index2-1249233.stderr
 delete mode 100644 tests/reference/runtime-test_list_index3-647a94a.json
 delete mode 100644 tests/reference/runtime-test_list_index3-647a94a.stderr
 delete mode 100644 tests/reference/runtime-test_list_index4-c31dfdb.json
 delete mode 100644 tests/reference/runtime-test_list_index4-c31dfdb.stderr
 delete mode 100644 tests/reference/runtime-test_str_01-50bdf2f.json
 delete mode 100644 tests/reference/runtime-test_str_01-50bdf2f.stdout
 delete mode 100644 tests/reference/runtime-test_str_02-c38ba27.json
 delete mode 100644 tests/reference/runtime-test_str_02-c38ba27.stdout
 delete mode 100644 tests/reference/tokens-comment1-2f8ab90.json
 delete mode 100644 tests/reference/tokens-comment1-2f8ab90.stdout
 delete mode 100644 tests/reference/tokens-comment2-b289dad.json
 delete mode 100644 tests/reference/tokens-comment2-b289dad.stdout
 delete mode 100644 tests/reference/tokens-docstring1-1355fbb.json
 delete mode 100644 tests/reference/tokens-docstring1-1355fbb.stdout
 delete mode 100644 tests/reference/tokens-indent1-0b243c5.json
 delete mode 100644 tests/reference/tokens-indent1-0b243c5.stderr
 delete mode 100644 tests/reference/tokens-indent1-290e858.json
 delete mode 100644 tests/reference/tokens-indent1-290e858.stdout
 delete mode 100644 tests/reference/tokens-indent2-a8438a0.json
 delete mode 100644 tests/reference/tokens-indent2-a8438a0.stderr
 delete mode 100644 tests/reference/tokens-indent2-e702789.json
 delete mode 100644 tests/reference/tokens-indent2-e702789.stdout
 delete mode 100644 tests/reference/tokens-indent3-0eaf3e6.json
 delete mode 100644 tests/reference/tokens-indent3-0eaf3e6.stderr
 delete mode 100644 tests/reference/tokens-numbers1-589063f.json
 delete mode 100644 tests/reference/tokens-numbers1-589063f.stdout
 delete mode 100644 tests/reference/tokens-paren1-94fa736.json
 delete mode 100644 tests/reference/tokens-paren1-94fa736.stderr
 delete mode 100644 tests/reference/tokens-paren2-0eef339.json
 delete mode 100644 tests/reference/tokens-paren2-0eef339.stderr
 delete mode 100644 tests/reference/tokens-paren3-0d47100.json
 delete mode 100644 tests/reference/tokens-paren3-0d47100.stderr
 delete mode 100644 tests/reference/tokens-paren4-5c2fc32.json
 delete mode 100644 tests/reference/tokens-paren4-5c2fc32.stderr
 delete mode 100644 tests/reference/tokens-symbols1-658c990.json
 delete mode 100644 tests/reference/tokens-symbols1-658c990.stdout
 delete mode 100644 tests/reference/tokens-test_literal-e20c024.json
 delete mode 100644 tests/reference/tokens-test_literal-e20c024.stderr
 delete mode 100644 tests/reference/tokens-test_literal-e20c024.stdout
 delete mode 100644 tests/reference/wat-bool1-234bcd1.json
 delete mode 100644 tests/reference/wat-bool1-234bcd1.stdout
 delete mode 100644 tests/reference/wat-expr14-5e0cb96.json
 delete mode 100644 tests/reference/wat-expr14-5e0cb96.stdout
 delete mode 100644 tests/reference/wat-expr2-8b17723.json
 delete mode 100644 tests/reference/wat-expr2-8b17723.stdout
 delete mode 100644 tests/reference/wat-expr9-f73afd1.json
 delete mode 100644 tests/reference/wat-expr9-f73afd1.stdout
 delete mode 100644 tests/reference/wat-loop1-e0046d4.json
 delete mode 100644 tests/reference/wat-loop1-e0046d4.stdout
 delete mode 100644 tests/reference/wat-print_str-385e953.json
 delete mode 100644 tests/reference/wat-print_str-385e953.stdout
```

Reply:
But nothing got commited.

Reply:
What is supposed to happen? I did this, but nothing got committed.

Reply:
Output:
```
(lp) kishan@kishan-IdeaPad-3-15IIL05:~/Desktop/lpython$ git commit -m "TEST: Update reference tests"
[replace 1c67c2f12] TEST: Update reference tests
 778 files changed, 107696 insertions(+)
 create mode 100644 tests/reference/asr-array_01_decl-39cf894.json
 create mode 100644 tests/reference/asr-array_01_decl-39cf894.stdout
 create mode 100644 tests/reference/asr-array_02_decl-e8f6874.json
 create mode 100644 tests/reference/asr-array_02_decl-e8f6874.stdout
 create mode 100644 tests/reference/asr-arrays_01-a617b64.json
 create mode 100644 tests/reference/asr-arrays_01-a617b64.stderr
 create mode 100644 tests/reference/asr-arrays_02-da94458.json
 create mode 100644 tests/reference/asr-arrays_02-da94458.stderr
 create mode 100644 tests/reference/asr-arrays_03-de2e952.json
 create mode 100644 tests/reference/asr-arrays_03-de2e952.stderr
 create mode 100644 tests/reference/asr-arrays_04-880407c.json
 create mode 100644 tests/reference/asr-arrays_04-880407c.stderr
 create mode 100644 tests/reference/asr-arrays_05-ec8fbd5.json
 create mode 100644 tests/reference/asr-arrays_05-ec8fbd5.stderr
 create mode 100644 tests/reference/asr-arrays_06-fbb09a3.json
 create mode 100644 tests/reference/asr-arrays_06-fbb09a3.stderr
 create mode 100644 tests/reference/asr-arrays_07-de430fd.json
 create mode 100644 tests/reference/asr-arrays_07-de430fd.stderr
 create mode 100644 tests/reference/asr-arrays_08-ba317a3.json
 create mode 100644 tests/reference/asr-arrays_08-ba317a3.stderr
 create mode 100644 tests/reference/asr-arrays_09-50ee586.json
 create mode 100644 tests/reference/asr-arrays_09-50ee586.stderr
 create mode 100644 tests/reference/asr-arrays_10-bc82d75.json
 create mode 100644 tests/reference/asr-arrays_10-bc82d75.stderr
 create mode 100644 tests/reference/asr-arrays_11-fc505b4.json
 create mode 100644 tests/reference/asr-arrays_11-fc505b4.stderr
 create mode 100644 tests/reference/asr-arrays_12-63d6f25.json
 create mode 100644 tests/reference/asr-arrays_12-63d6f25.stderr
 create mode 100644 tests/reference/asr-arrays_13-b5fcc7e.json
 create mode 100644 tests/reference/asr-arrays_13-b5fcc7e.stderr
 create mode 100644 tests/reference/asr-arrays_14-78be00e.json
 create mode 100644 tests/reference/asr-arrays_14-78be00e.stderr
 create mode 100644 tests/reference/asr-assert1-1ce92ea.json
 create mode 100644 tests/reference/asr-assert1-1ce92ea.stdout
 create mode 100644 tests/reference/asr-assign1-886f049.json
 create mode 100644 tests/reference/asr-assign1-886f049.stdout
 create mode 100644 tests/reference/asr-assign2-8d1a2ee.json
 create mode 100644 tests/reference/asr-assign2-8d1a2ee.stdout
 create mode 100644 tests/reference/asr-bindc_01-6d521a9.json
 create mode 100644 tests/reference/asr-bindc_01-6d521a9.stdout
 create mode 100644 tests/reference/asr-bindc_01-f761165.json
 create mode 100644 tests/reference/asr-bindc_01-f761165.stderr
 create mode 100644 tests/reference/asr-bindc_02-5092d8e.json
 create mode 100644 tests/reference/asr-bindc_02-5092d8e.stderr
 create mode 100644 tests/reference/asr-bindc_02-bc1a7ea.json
 create mode 100644 tests/reference/asr-bindc_02-bc1a7ea.stdout
 create mode 100644 tests/reference/asr-bindc_03-95dbba7.json
 create mode 100644 tests/reference/asr-bindc_03-95dbba7.stderr
 create mode 100644 tests/reference/asr-bindc_04-06bd800.json
 create mode 100644 tests/reference/asr-bindc_04-06bd800.stderr
 create mode 100644 tests/reference/asr-bindc_10e-8b10394.json
 create mode 100644 tests/reference/asr-bindc_10e-8b10394.stderr
 create mode 100644 tests/reference/asr-c_interop1-cf2e9b4.json
 create mode 100644 tests/reference/asr-c_interop1-cf2e9b4.stdout
 create mode 100644 tests/reference/asr-callback_01-df40fd5.json
 create mode 100644 tests/reference/asr-callback_01-df40fd5.stdout
 create mode 100644 tests/reference/asr-cast-435c233.json
 create mode 100644 tests/reference/asr-cast-435c233.stdout
 create mode 100644 tests/reference/asr-complex1-f26c460.json
 create mode 100644 tests/reference/asr-complex1-f26c460.stdout
 create mode 100644 tests/reference/asr-const_01-af8289b.json
 create mode 100644 tests/reference/asr-const_01-af8289b.stderr
 create mode 100644 tests/reference/asr-const_02-fce29b7.json
 create mode 100644 tests/reference/asr-const_02-fce29b7.stderr
 create mode 100644 tests/reference/asr-constants1-5828e8a.json
 create mode 100644 tests/reference/asr-constants1-5828e8a.stdout
 create mode 100644 tests/reference/asr-cptr_01-4e660f1.json
 create mode 100644 tests/reference/asr-cptr_01-4e660f1.stderr
 create mode 100644 tests/reference/asr-dictionary1-a105a36.json
 create mode 100644 tests/reference/asr-dictionary1-a105a36.stdout
 create mode 100644 tests/reference/asr-doconcurrentloop_01-3fdc189.json
 create mode 100644 tests/reference/asr-doconcurrentloop_01-3fdc189.stdout
 create mode 100644 tests/reference/asr-elemental_01-b58df26.json
 create mode 100644 tests/reference/asr-elemental_01-b58df26.stdout
 create mode 100644 tests/reference/asr-enum_01-30e1b4a.json
 create mode 100644 tests/reference/asr-enum_01-30e1b4a.stderr
 create mode 100644 tests/reference/asr-enum_02-54656c5.json
 create mode 100644 tests/reference/asr-enum_02-54656c5.stderr
 create mode 100644 tests/reference/asr-expr1-8df2d66.json
 create mode 100644 tests/reference/asr-expr1-8df2d66.stdout
 create mode 100644 tests/reference/asr-expr10-efcbb1b.json
 create mode 100644 tests/reference/asr-expr10-efcbb1b.stdout
 create mode 100644 tests/reference/asr-expr11-9b91d35.json
 create mode 100644 tests/reference/asr-expr11-9b91d35.stdout
 create mode 100644 tests/reference/asr-expr12-5c5b71e.json
 create mode 100644 tests/reference/asr-expr12-5c5b71e.stdout
 create mode 100644 tests/reference/asr-expr13-81bdb5a.json
 create mode 100644 tests/reference/asr-expr13-81bdb5a.stdout
 create mode 100644 tests/reference/asr-expr16-a3dc453.json
 create mode 100644 tests/reference/asr-expr16-a3dc453.stderr
 create mode 100644 tests/reference/asr-expr2-2e78a12.json
 create mode 100644 tests/reference/asr-expr2-2e78a12.stdout
 create mode 100644 tests/reference/asr-expr4-cef6743.json
 create mode 100644 tests/reference/asr-expr4-cef6743.stdout
 create mode 100644 tests/reference/asr-expr5-645ffcc.json
 create mode 100644 tests/reference/asr-expr5-645ffcc.stdout
 create mode 100644 tests/reference/asr-expr6-368e5ed.json
 create mode 100644 tests/reference/asr-expr6-368e5ed.stdout
 create mode 100644 tests/reference/asr-expr7-480ba2f.json
 create mode 100644 tests/reference/asr-expr7-480ba2f.stderr
 create mode 100644 tests/reference/asr-expr7-480ba2f.stdout
 create mode 100644 tests/reference/asr-expr8-6beda60.json
 create mode 100644 tests/reference/asr-expr8-6beda60.stdout
 create mode 100644 tests/reference/asr-expr9-814e4bc.json
 create mode 100644 tests/reference/asr-expr9-814e4bc.stdout
 create mode 100644 tests/reference/asr-expr_01-211000e.json
 create mode 100644 tests/reference/asr-expr_01-211000e.stdout
 create mode 100644 tests/reference/asr-expr_01-a0d4829.json
 create mode 100644 tests/reference/asr-expr_01-a0d4829.stdout
 create mode 100644 tests/reference/asr-expr_05-3a37324.json
 create mode 100644 tests/reference/asr-expr_05-3a37324.stdout
 create mode 100644 tests/reference/asr-expr_07-7742668.json
 create mode 100644 tests/reference/asr-expr_07-7742668.stdout
 create mode 100644 tests/reference/asr-expr_09-f3e89c8.json
 create mode 100644 tests/reference/asr-expr_09-f3e89c8.stdout
 create mode 100644 tests/reference/asr-expr_10-d39708c.json
 create mode 100644 tests/reference/asr-expr_10-d39708c.stdout
 create mode 100644 tests/reference/asr-expr_12-6769be0.json
 create mode 100644 tests/reference/asr-expr_12-6769be0.stdout
 create mode 100644 tests/reference/asr-expr_14-f2bd343.json
 create mode 100644 tests/reference/asr-expr_14-f2bd343.stdout
 create mode 100644 tests/reference/asr-func_01-d87aa4a.json
 create mode 100644 tests/reference/asr-func_01-d87aa4a.stderr
 create mode 100644 tests/reference/asr-func_02-b439474.json
 create mode 100644 tests/reference/asr-func_02-b439474.stderr
 create mode 100644 tests/reference/asr-func_03-cd744a0.json
 create mode 100644 tests/reference/asr-func_03-cd744a0.stderr
 create mode 100644 tests/reference/asr-func_04-eef2656.json
 create mode 100644 tests/reference/asr-func_04-eef2656.stderr
 create mode 100644 tests/reference/asr-func_05-c22b921.json
 create mode 100644 tests/reference/asr-func_05-c22b921.stderr
 create mode 100644 tests/reference/asr-func_06-62e738c.json
 create mode 100644 tests/reference/asr-func_06-62e738c.stderr
 create mode 100644 tests/reference/asr-func_07-4a8c076.json
 create mode 100644 tests/reference/asr-func_07-4a8c076.stderr
 create mode 100644 tests/reference/asr-func_08-c137415.json
 create mode 100644 tests/reference/asr-func_08-c137415.stderr
 create mode 100644 tests/reference/asr-func_inline_01-56af272.json
 create mode 100644 tests/reference/asr-func_inline_01-56af272.stdout
 create mode 100644 tests/reference/asr-generics_01-d616074.json
 create mode 100644 tests/reference/asr-generics_01-d616074.stdout
 create mode 100644 tests/reference/asr-generics_02-e2ea5c9.json
 create mode 100644 tests/reference/asr-generics_02-e2ea5c9.stdout
 create mode 100644 tests/reference/asr-generics_array_01-682b1b2.json
 create mode 100644 tests/reference/asr-generics_array_01-682b1b2.stdout
 create mode 100644 tests/reference/asr-generics_array_02-22c8dc1.json
 create mode 100644 tests/reference/asr-generics_array_02-22c8dc1.stdout
 create mode 100644 tests/reference/asr-generics_array_03-fb3706c.json
 create mode 100644 tests/reference/asr-generics_array_03-fb3706c.stdout
 create mode 100644 tests/reference/asr-generics_error_01-1e05cd6.json
 create mode 100644 tests/reference/asr-generics_error_01-1e05cd6.stderr
 create mode 100644 tests/reference/asr-generics_error_02-d614928.json
 create mode 100644 tests/reference/asr-generics_error_02-d614928.stderr
 create mode 100644 tests/reference/asr-generics_error_03-208d10d.json
 create mode 100644 tests/reference/asr-generics_error_03-208d10d.stderr
 create mode 100644 tests/reference/asr-generics_list_01-39c4044.json
 create mode 100644 tests/reference/asr-generics_list_01-39c4044.stdout
 create mode 100644 tests/reference/asr-global_scope1-354e217.json
 create mode 100644 tests/reference/asr-global_scope1-354e217.stdout
 create mode 100644 tests/reference/asr-global_syms_01-273906f.json
 create mode 100644 tests/reference/asr-global_syms_01-273906f.stdout
 create mode 100644 tests/reference/asr-intent_01-66824bc.json
 create mode 100644 tests/reference/asr-intent_01-66824bc.stdout
 create mode 100644 tests/reference/asr-kwargs_01_error-ab9530d.json
 create mode 100644 tests/reference/asr-kwargs_01_error-ab9530d.stderr
 create mode 100644 tests/reference/asr-kwargs_02_error-7c91f8f.json
 create mode 100644 tests/reference/asr-kwargs_02_error-7c91f8f.stderr
 create mode 100644 tests/reference/asr-kwargs_03_error-a68cc46.json
 create mode 100644 tests/reference/asr-kwargs_03_error-a68cc46.stderr
 create mode 100644 tests/reference/asr-lambda_01-1ec3e01.json
 create mode 100644 tests/reference/asr-lambda_01-1ec3e01.stderr
 create mode 100644 tests/reference/asr-list1-770ba33.json
 create mode 100644 tests/reference/asr-list1-770ba33.stdout
 create mode 100644 tests/reference/asr-loop1-10d3109.json
 create mode 100644 tests/reference/asr-loop1-10d3109.stdout
 create mode 100644 tests/reference/asr-loop3-a579196.json
 create mode 100644 tests/reference/asr-loop3-a579196.stdout
 create mode 100644 tests/reference/asr-loop4-3d3216e.json
 create mode 100644 tests/reference/asr-loop4-3d3216e.stdout
 create mode 100644 tests/reference/asr-loop_01-e7aa06a.json
 create mode 100644 tests/reference/asr-loop_01-e7aa06a.stderr
 create mode 100644 tests/reference/asr-loop_02-5741651.json
 create mode 100644 tests/reference/asr-loop_02-5741651.stderr
 create mode 100644 tests/reference/asr-loop_03-401ab45.json
 create mode 100644 tests/reference/asr-loop_03-401ab45.stderr
 create mode 100644 tests/reference/asr-modules_02-ec92e6f.json
 create mode 100644 tests/reference/asr-modules_02-ec92e6f.stdout
 create mode 100644 tests/reference/asr-print_02-afbe092.json
 create mode 100644 tests/reference/asr-print_02-afbe092.stdout
 create mode 100644 tests/reference/asr-print_list_tuple_03-9de3736.json
 create mode 100644 tests/reference/asr-print_list_tuple_03-9de3736.stdout
 create mode 100644 tests/reference/asr-set1-b7b913a.json
 create mode 100644 tests/reference/asr-set1-b7b913a.stdout
 create mode 100644 tests/reference/asr-string_01-78629c4.json
 create mode 100644 tests/reference/asr-string_01-78629c4.stderr
 create mode 100644 tests/reference/asr-string_02-499c9ff.json
 create mode 100644 tests/reference/asr-string_02-499c9ff.stderr
 create mode 100644 tests/reference/asr-structs_01-0893e35.json
 create mode 100644 tests/reference/asr-structs_01-0893e35.stderr
 create mode 100644 tests/reference/asr-structs_01-66dc2c9.json
 create mode 100644 tests/reference/asr-structs_01-66dc2c9.stdout
 create mode 100644 tests/reference/asr-structs_01-be14d49.json
 create mode 100644 tests/reference/asr-structs_01-be14d49.stdout
 create mode 100644 tests/reference/asr-structs_02-2ab459a.json
 create mode 100644 tests/reference/asr-structs_02-2ab459a.stdout
 create mode 100644 tests/reference/asr-structs_02-f95782c.json
 create mode 100644 tests/reference/asr-structs_02-f95782c.stderr
 create mode 100644 tests/reference/asr-structs_03-0cef911.json
 create mode 100644 tests/reference/asr-structs_03-0cef911.stdout
 create mode 100644 tests/reference/asr-structs_03-754fb64.json
 create mode 100644 tests/reference/asr-structs_03-754fb64.stderr
 create mode 100644 tests/reference/asr-structs_04-387747b.json
 create mode 100644 tests/reference/asr-structs_04-387747b.stdout
 create mode 100644 tests/reference/asr-structs_04-7b864bc.json
 create mode 100644 tests/reference/asr-structs_04-7b864bc.stderr
 create mode 100644 tests/reference/asr-structs_05-fa98307.json
 create mode 100644 tests/reference/asr-structs_05-fa98307.stdout
 create mode 100644 tests/reference/asr-structs_06-6e14537.json
 create mode 100644 tests/reference/asr-structs_06-6e14537.stderr
 create mode 100644 tests/reference/asr-structs_07-83694b7.json
 create mode 100644 tests/reference/asr-structs_07-83694b7.stderr
 create mode 100644 tests/reference/asr-structs_08-fa4dbf0.json
 create mode 100644 tests/reference/asr-structs_08-fa4dbf0.stderr
 create mode 100644 tests/reference/asr-structs_09-f3ffe08.json
 create mode 100644 tests/reference/asr-structs_09-f3ffe08.stderr
 create mode 100644 tests/reference/asr-structs_10-cb8a283.json
 create mode 100644 tests/reference/asr-structs_10-cb8a283.stderr
 create mode 100644 tests/reference/asr-structs_16-44de89a.json
 create mode 100644 tests/reference/asr-structs_16-44de89a.stdout
 create mode 100644 tests/reference/asr-subscript1-1acfc19.json
 create mode 100644 tests/reference/asr-subscript1-1acfc19.stdout
 create mode 100644 tests/reference/asr-test_annassign_01-2f18669.json
 create mode 100644 tests/reference/asr-test_annassign_01-2f18669.stderr
 create mode 100644 tests/reference/asr-test_annassign_02-accf6db.json
 create mode 100644 tests/reference/asr-test_annassign_02-accf6db.stderr
 create mode 100644 tests/reference/asr-test_annassign_03-f8c6160.json
 create mode 100644 tests/reference/asr-test_annassign_03-f8c6160.stderr
 create mode 100644 tests/reference/asr-test_annassign_type_mismatch-7dac7be.json
 create mode 100644 tests/reference/asr-test_annassign_type_mismatch-7dac7be.stderr
 create mode 100644 tests/reference/asr-test_annassign_type_mismatch2-fc883f7.json
 create mode 100644 tests/reference/asr-test_annassign_type_mismatch2-fc883f7.stderr
 create mode 100644 tests/reference/asr-test_append_type_mismatch-030e9c7.json
 create mode 100644 tests/reference/asr-test_append_type_mismatch-030e9c7.stderr
 create mode 100644 tests/reference/asr-test_assign1-a94d41e.json
 create mode 100644 tests/reference/asr-test_assign1-a94d41e.stderr
 create mode 100644 tests/reference/asr-test_assign2-fa29029.json
 create mode 100644 tests/reference/asr-test_assign2-fa29029.stderr
 create mode 100644 tests/reference/asr-test_assign3-cc6fc9d.json
 create mode 100644 tests/reference/asr-test_assign3-cc6fc9d.stderr
 create mode 100644 tests/reference/asr-test_assign4-a2136e0.json
 create mode 100644 tests/reference/asr-test_assign4-a2136e0.stderr
 create mode 100644 tests/reference/asr-test_assign5-694a98f.json
 create mode 100644 tests/reference/asr-test_assign5-694a98f.stderr
 create mode 100644 tests/reference/asr-test_assign6-05cd64f.json
 create mode 100644 tests/reference/asr-test_assign6-05cd64f.stderr
 create mode 100644 tests/reference/asr-test_assign7-beebac3.json
 create mode 100644 tests/reference/asr-test_assign7-beebac3.stderr
 create mode 100644 tests/reference/asr-test_assign8-4b26e63.json
 create mode 100644 tests/reference/asr-test_assign8-4b26e63.stderr
 create mode 100644 tests/reference/asr-test_assign9-51278b8.json
 create mode 100644 tests/reference/asr-test_assign9-51278b8.stderr
 create mode 100644 tests/reference/asr-test_async-361297c.json
 create mode 100644 tests/reference/asr-test_async-361297c.stderr
 create mode 100644 tests/reference/asr-test_binop1-50b63f6.json
 create mode 100644 tests/reference/asr-test_binop1-50b63f6.stderr
 create mode 100644 tests/reference/asr-test_binop2-e5749af.json
 create mode 100644 tests/reference/asr-test_binop2-e5749af.stderr
 create mode 100644 tests/reference/asr-test_binop3-a67201d.json
 create mode 100644 tests/reference/asr-test_binop3-a67201d.stderr
 create mode 100644 tests/reference/asr-test_bit_length-da3a264.json
 create mode 100644 tests/reference/asr-test_bit_length-da3a264.stderr
 create mode 100644 tests/reference/asr-test_bitwise_on_complex-dd9568b.json
 create mode 100644 tests/reference/asr-test_bitwise_on_complex-dd9568b.stderr
 create mode 100644 tests/reference/asr-test_bitwise_on_float-2e09b30.json
 create mode 100644 tests/reference/asr-test_bitwise_on_float-2e09b30.stderr
 create mode 100644 tests/reference/asr-test_bool_binop-f856ef0.json
 create mode 100644 tests/reference/asr-test_bool_binop-f856ef0.stdout
 create mode 100644 tests/reference/asr-test_builtin-aa64615.json
 create mode 100644 tests/reference/asr-test_builtin-aa64615.stdout
 create mode 100644 tests/reference/asr-test_builtin_abs-c74d2c9.json
 create mode 100644 tests/reference/asr-test_builtin_abs-c74d2c9.stdout
 create mode 100644 tests/reference/asr-test_builtin_bin-52ba9fa.json
 create mode 100644 tests/reference/asr-test_builtin_bin-52ba9fa.stdout
 create mode 100644 tests/reference/asr-test_builtin_bool-330223a.json
 create mode 100644 tests/reference/asr-test_builtin_bool-330223a.stdout
 create mode 100644 tests/reference/asr-test_builtin_float-20601dd.json
 create mode 100644 tests/reference/asr-test_builtin_float-20601dd.stdout
 create mode 100644 tests/reference/asr-test_builtin_hex-64bd268.json
 create mode 100644 tests/reference/asr-test_builtin_hex-64bd268.stdout
 create mode 100644 tests/reference/asr-test_builtin_int-8f88fdc.json
 create mode 100644 tests/reference/asr-test_builtin_int-8f88fdc.stdout
 create mode 100644 tests/reference/asr-test_builtin_len-55b0dec.json
 create mode 100644 tests/reference/asr-test_builtin_len-55b0dec.stdout
 create mode 100644 tests/reference/asr-test_builtin_oct-20b9066.json
 create mode 100644 tests/reference/asr-test_builtin_oct-20b9066.stdout
 create mode 100644 tests/reference/asr-test_builtin_pow-f02fcda.json
 create mode 100644 tests/reference/asr-test_builtin_pow-f02fcda.stderr
 create mode 100644 tests/reference/asr-test_builtin_pow-f02fcda.stdout
 create mode 100644 tests/reference/asr-test_builtin_round-7417a21.json
 create mode 100644 tests/reference/asr-test_builtin_round-7417a21.stdout
 create mode 100644 tests/reference/asr-test_builtin_str-580e920.json
 create mode 100644 tests/reference/asr-test_builtin_str-580e920.stdout
 create mode 100644 tests/reference/asr-test_c_interop_01-e374f43.json
 create mode 100644 tests/reference/asr-test_c_interop_01-e374f43.stdout
 create mode 100644 tests/reference/asr-test_chr-91f4987.json
 create mode 100644 tests/reference/asr-test_chr-91f4987.stderr
 create mode 100644 tests/reference/asr-test_compare_01-c594bbe.json
 create mode 100644 tests/reference/asr-test_compare_01-c594bbe.stderr
 create mode 100644 tests/reference/asr-test_compare_02-bf2c76e.json
 create mode 100644 tests/reference/asr-test_compare_02-bf2c76e.stderr
 create mode 100644 tests/reference/asr-test_complex_01-a6def58.json
 create mode 100644 tests/reference/asr-test_complex_01-a6def58.stdout
 create mode 100644 tests/reference/asr-test_complex_02-782ba2d.json
 create mode 100644 tests/reference/asr-test_complex_02-782ba2d.stdout
 create mode 100644 tests/reference/asr-test_dict1-a45a075.json
 create mode 100644 tests/reference/asr-test_dict1-a45a075.stderr
 create mode 100644 tests/reference/asr-test_dict10-8c0beff.json
 create mode 100644 tests/reference/asr-test_dict10-8c0beff.stderr
 create mode 100644 tests/reference/asr-test_dict11-2ab4e6c.json
 create mode 100644 tests/reference/asr-test_dict11-2ab4e6c.stderr
 create mode 100644 tests/reference/asr-test_dict12-7c74d08.json
 create mode 100644 tests/reference/asr-test_dict12-7c74d08.stderr
 create mode 100644 tests/reference/asr-test_dict13-683b290.json
 create mode 100644 tests/reference/asr-test_dict13-683b290.stderr
 create mode 100644 tests/reference/asr-test_dict2-4587f02.json
 create mode 100644 tests/reference/asr-test_dict2-4587f02.stderr
 create mode 100644 tests/reference/asr-test_dict3-d28f38f.json
 create mode 100644 tests/reference/asr-test_dict3-d28f38f.stderr
 create mode 100644 tests/reference/asr-test_dict4-39489fa.json
 create mode 100644 tests/reference/asr-test_dict4-39489fa.stderr
 create mode 100644 tests/reference/asr-test_dict5-c436d37.json
 create mode 100644 tests/reference/asr-test_dict5-c436d37.stderr
 create mode 100644 tests/reference/asr-test_dict6-95e3b14.json
 create mode 100644 tests/reference/asr-test_dict6-95e3b14.stderr
 create mode 100644 tests/reference/asr-test_dict7-1415e14.json
 create mode 100644 tests/reference/asr-test_dict7-1415e14.stderr
 create mode 100644 tests/reference/asr-test_dict8-d960ce0.json
 create mode 100644 tests/reference/asr-test_dict8-d960ce0.stderr
 create mode 100644 tests/reference/asr-test_dict9-907bda7.json
 create mode 100644 tests/reference/asr-test_dict9-907bda7.stderr
 create mode 100644 tests/reference/asr-test_end_sep_keywords-2226a67.json
 create mode 100644 tests/reference/asr-test_end_sep_keywords-2226a67.stdout
 create mode 100644 tests/reference/asr-test_float_semantic_error-58c0c90.json
 create mode 100644 tests/reference/asr-test_float_semantic_error-58c0c90.stderr
 create mode 100644 tests/reference/asr-test_for2-e5d893f.json
 create mode 100644 tests/reference/asr-test_for2-e5d893f.stderr
 create mode 100644 tests/reference/asr-test_func_args-a898a72.json
 create mode 100644 tests/reference/asr-test_func_args-a898a72.stderr
 create mode 100644 tests/reference/asr-test_import_01-b859c43.json
 create mode 100644 tests/reference/asr-test_import_01-b859c43.stderr
 create mode 100644 tests/reference/asr-test_import_02-55b47fa.json
 create mode 100644 tests/reference/asr-test_import_02-55b47fa.stderr
 create mode 100644 tests/reference/asr-test_int_semantic_error-44fe25e.json
 create mode 100644 tests/reference/asr-test_int_semantic_error-44fe25e.stderr
 create mode 100644 tests/reference/asr-test_len1-839a615.json
 create mode 100644 tests/reference/asr-test_len1-839a615.stderr
 create mode 100644 tests/reference/asr-test_list1-73fd538.json
 create mode 100644 tests/reference/asr-test_list1-73fd538.stderr
 create mode 100644 tests/reference/asr-test_list2-10ffdd7.json
 create mode 100644 tests/reference/asr-test_list2-10ffdd7.stderr
 create mode 100644 tests/reference/asr-test_list3-5f4d2a8.json
 create mode 100644 tests/reference/asr-test_list3-5f4d2a8.stderr
 create mode 100644 tests/reference/asr-test_list4-d18a7e3.json
 create mode 100644 tests/reference/asr-test_list4-d18a7e3.stderr
 create mode 100644 tests/reference/asr-test_list_concat-41d186f.json
 create mode 100644 tests/reference/asr-test_list_concat-41d186f.stderr
 create mode 100644 tests/reference/asr-test_list_count-4b42498.json
 create mode 100644 tests/reference/asr-test_list_count-4b42498.stderr
 create mode 100644 tests/reference/asr-test_list_slicing-984fbf0.json
 create mode 100644 tests/reference/asr-test_list_slicing-984fbf0.stderr
 create mode 100644 tests/reference/asr-test_max_min-3c2fc51.json
 create mode 100644 tests/reference/asr-test_max_min-3c2fc51.stdout
 create mode 100644 tests/reference/asr-test_numpy_03-e600a49.json
 create mode 100644 tests/reference/asr-test_numpy_03-e600a49.stdout
 create mode 100644 tests/reference/asr-test_numpy_04-ecbb614.json
 create mode 100644 tests/reference/asr-test_numpy_04-ecbb614.stdout
 create mode 100644 tests/reference/asr-test_operator_01-0f232bf.json
 create mode 100644 tests/reference/asr-test_operator_01-0f232bf.stderr
 create mode 100644 tests/reference/asr-test_ord-316ce61.json
 create mode 100644 tests/reference/asr-test_ord-316ce61.stderr
 create mode 100644 tests/reference/asr-test_pointer_types-1bf0d01.json
 create mode 100644 tests/reference/asr-test_pointer_types-1bf0d01.stderr
 create mode 100644 tests/reference/asr-test_pow-3f5d550.json
 create mode 100644 tests/reference/asr-test_pow-3f5d550.stderr
 create mode 100644 tests/reference/asr-test_pow-3f5d550.stdout
 create mode 100644 tests/reference/asr-test_pow1-581ef42.json
 create mode 100644 tests/reference/asr-test_pow1-581ef42.stderr
 create mode 100644 tests/reference/asr-test_pow2-0dcbd7d.json
 create mode 100644 tests/reference/asr-test_pow2-0dcbd7d.stderr
 create mode 100644 tests/reference/asr-test_pow3-79749ed.json
 create mode 100644 tests/reference/asr-test_pow3-79749ed.stderr
 create mode 100644 tests/reference/asr-test_pow4-ef60978.json
 create mode 100644 tests/reference/asr-test_pow4-ef60978.stderr
 create mode 100644 tests/reference/asr-test_print1-f1f36f1.json
 create mode 100644 tests/reference/asr-test_print1-f1f36f1.stderr
 create mode 100644 tests/reference/asr-test_print2-64acb15.json
 create mode 100644 tests/reference/asr-test_print2-64acb15.stderr
 create mode 100644 tests/reference/asr-test_return1-2603c9e.json
 create mode 100644 tests/reference/asr-test_return1-2603c9e.stderr
 create mode 100644 tests/reference/asr-test_return2-add829b.json
 create mode 100644 tests/reference/asr-test_return2-add829b.stderr
 create mode 100644 tests/reference/asr-test_set1-11379c7.json
 create mode 100644 tests/reference/asr-test_set1-11379c7.stderr
 create mode 100644 tests/reference/asr-test_set2-d91a6f0.json
 create mode 100644 tests/reference/asr-test_set2-d91a6f0.stderr
 create mode 100644 tests/reference/asr-test_set3-f9bbffb.json
 create mode 100644 tests/reference/asr-test_set3-f9bbffb.stderr
 create mode 100644 tests/reference/asr-test_set4-53fea39.json
 create mode 100644 tests/reference/asr-test_set4-53fea39.stderr
 create mode 100644 tests/reference/asr-test_set_indexing-a1c21b8.json
 create mode 100644 tests/reference/asr-test_set_indexing-a1c21b8.stderr
 create mode 100644 tests/reference/asr-test_str_capitalize-9f8dfa9.json
 create mode 100644 tests/reference/asr-test_str_capitalize-9f8dfa9.stderr
 create mode 100644 tests/reference/asr-test_str_indexing-b200a4e.json
 create mode 100644 tests/reference/asr-test_str_indexing-b200a4e.stderr
 create mode 100644 tests/reference/asr-test_str_slicing-eca1381.json
 create mode 100644 tests/reference/asr-test_str_slicing-eca1381.stderr
 create mode 100644 tests/reference/asr-test_str_slicing2-2f07e9a.json
 create mode 100644 tests/reference/asr-test_str_slicing2-2f07e9a.stderr
 create mode 100644 tests/reference/asr-test_str_slicing3-fe6a03c.json
 create mode 100644 tests/reference/asr-test_str_slicing3-fe6a03c.stderr
 create mode 100644 tests/reference/asr-test_str_slicing4-a0c7a69.json
 create mode 100644 tests/reference/asr-test_str_slicing4-a0c7a69.stderr
 create mode 100644 tests/reference/asr-test_str_to_int-61553e7.json
 create mode 100644 tests/reference/asr-test_str_to_int-61553e7.stderr
 create mode 100644 tests/reference/asr-test_tuple1-7abe88f.json
 create mode 100644 tests/reference/asr-test_tuple1-7abe88f.stderr
 create mode 100644 tests/reference/asr-test_tuple2-b046610.json
 create mode 100644 tests/reference/asr-test_tuple2-b046610.stderr
 create mode 100644 tests/reference/asr-test_tuple3-19fa47f.json
 create mode 100644 tests/reference/asr-test_tuple3-19fa47f.stderr
 create mode 100644 tests/reference/asr-test_type_mismatch_01-09e8af3.json
 create mode 100644 tests/reference/asr-test_type_mismatch_01-09e8af3.stderr
 create mode 100644 tests/reference/asr-test_unary_op_03-e799eae.json
 create mode 100644 tests/reference/asr-test_unary_op_03-e799eae.stdout
 create mode 100644 tests/reference/asr-test_unsupported_type-0d813dd.json
 create mode 100644 tests/reference/asr-test_unsupported_type-0d813dd.stderr
 create mode 100644 tests/reference/asr-test_zero_division-3dd84e8.json
 create mode 100644 tests/reference/asr-test_zero_division-3dd84e8.stderr
 create mode 100644 tests/reference/asr-test_zero_division2-d84989f.json
 create mode 100644 tests/reference/asr-test_zero_division2-d84989f.stderr
 create mode 100644 tests/reference/asr-test_zero_division3-29efb9e.json
 create mode 100644 tests/reference/asr-test_zero_division3-29efb9e.stderr
 create mode 100644 tests/reference/asr-test_zero_division4-bf4af64.json
 create mode 100644 tests/reference/asr-test_zero_division4-bf4af64.stderr
 create mode 100644 tests/reference/asr-tuple1-09972ab.json
 create mode 100644 tests/reference/asr-tuple1-09972ab.stdout
 create mode 100644 tests/reference/asr-type_mismatch-61052c7.json
 create mode 100644 tests/reference/asr-type_mismatch-61052c7.stderr
 create mode 100644 tests/reference/asr-unsigned_01-892b178.json
 create mode 100644 tests/reference/asr-unsigned_01-892b178.stderr
 create mode 100644 tests/reference/asr-unsigned_02-6563e58.json
 create mode 100644 tests/reference/asr-unsigned_02-6563e58.stderr
 create mode 100644 tests/reference/asr-unsigned_03-f785652.json
 create mode 100644 tests/reference/asr-unsigned_03-f785652.stderr
 create mode 100644 tests/reference/asr-unsigned_04-c856d3a.json
 create mode 100644 tests/reference/asr-unsigned_04-c856d3a.stderr
 create mode 100644 tests/reference/asr-vec_01-66ac423.json
 create mode 100644 tests/reference/asr-vec_01-66ac423.stdout
 create mode 100644 tests/reference/asr_json-modules_02-53952e6.json
 create mode 100644 tests/reference/asr_json-modules_02-53952e6.stdout
 create mode 100644 tests/reference/ast-assert1-b0154ee.json
 create mode 100644 tests/reference/ast-assert1-b0154ee.stdout
 create mode 100644 tests/reference/ast-assign1-2a4c9ed.json
 create mode 100644 tests/reference/ast-assign1-2a4c9ed.stdout
 create mode 100644 tests/reference/ast-complex1-800b4bb.json
 create mode 100644 tests/reference/ast-complex1-800b4bb.stdout
 create mode 100644 tests/reference/ast-constants1-91cb6ff.json
 create mode 100644 tests/reference/ast-constants1-91cb6ff.stdout
 create mode 100644 tests/reference/ast-dictionary1-1a7e00a.json
 create mode 100644 tests/reference/ast-dictionary1-1a7e00a.stdout
 create mode 100644 tests/reference/ast-doconcurrentloop_01-ed7017b.json
 create mode 100644 tests/reference/ast-doconcurrentloop_01-ed7017b.stdout
 create mode 100644 tests/reference/ast-ellipsis1-4f6c4dd.json
 create mode 100644 tests/reference/ast-ellipsis1-4f6c4dd.stdout
 create mode 100644 tests/reference/ast-expr1-1e8f7b1.json
 create mode 100644 tests/reference/ast-expr1-1e8f7b1.stdout
 create mode 100644 tests/reference/ast-expr10-a8d646d.json
 create mode 100644 tests/reference/ast-expr10-a8d646d.stdout
 create mode 100644 tests/reference/ast-expr11-1d29f78.json
 create mode 100644 tests/reference/ast-expr11-1d29f78.stdout
 create mode 100644 tests/reference/ast-expr12-adaecda.json
 create mode 100644 tests/reference/ast-expr12-adaecda.stdout
 create mode 100644 tests/reference/ast-expr13-c35ace1.json
 create mode 100644 tests/reference/ast-expr13-c35ace1.stdout
 create mode 100644 tests/reference/ast-expr2-6642d4a.json
 create mode 100644 tests/reference/ast-expr2-6642d4a.stdout
 create mode 100644 tests/reference/ast-expr4-49316cb.json
 create mode 100644 tests/reference/ast-expr4-49316cb.stdout
 create mode 100644 tests/reference/ast-expr5-bbc6e71.json
 create mode 100644 tests/reference/ast-expr5-bbc6e71.stdout
 create mode 100644 tests/reference/ast-expr6-0b12a67.json
 create mode 100644 tests/reference/ast-expr6-0b12a67.stdout
 create mode 100644 tests/reference/ast-expr7-fe52776.json
 create mode 100644 tests/reference/ast-expr7-fe52776.stdout
 create mode 100644 tests/reference/ast-expr8-7db6b28.json
 create mode 100644 tests/reference/ast-expr8-7db6b28.stdout
 create mode 100644 tests/reference/ast-expr9-d184496.json
 create mode 100644 tests/reference/ast-expr9-d184496.stdout
 create mode 100644 tests/reference/ast-expr_01-d0927f9.json
 create mode 100644 tests/reference/ast-expr_01-d0927f9.stdout
 create mode 100644 tests/reference/ast-global1-b2690cf.json
 create mode 100644 tests/reference/ast-global1-b2690cf.stdout
 create mode 100644 tests/reference/ast-global_scope1-1d68a6c.json
 create mode 100644 tests/reference/ast-global_scope1-1d68a6c.stdout
 create mode 100644 tests/reference/ast-list1-9ce2da0.json
 create mode 100644 tests/reference/ast-list1-9ce2da0.stdout
 create mode 100644 tests/reference/ast-loop1-194a137.json
 create mode 100644 tests/reference/ast-loop1-194a137.stdout
 create mode 100644 tests/reference/ast-loop3-f7e0393.json
 create mode 100644 tests/reference/ast-loop3-f7e0393.stdout
 create mode 100644 tests/reference/ast-prefix_string_01-cf221fd.json
 create mode 100644 tests/reference/ast-prefix_string_01-cf221fd.stderr
 create mode 100644 tests/reference/ast-prefix_string_02-3d530b2.json
 create mode 100644 tests/reference/ast-prefix_string_02-3d530b2.stderr
 create mode 100644 tests/reference/ast-set1-ebd6ee0.json
 create mode 100644 tests/reference/ast-set1-ebd6ee0.stdout
 create mode 100644 tests/reference/ast-subscript1-bd5584b.json
 create mode 100644 tests/reference/ast-subscript1-bd5584b.stdout
 create mode 100644 tests/reference/ast-tuple1-2fb5396.json
 create mode 100644 tests/reference/ast-tuple1-2fb5396.stdout
 create mode 100644 tests/reference/ast_new-async1-b3d07ed.json
 create mode 100644 tests/reference/ast_new-async1-b3d07ed.stdout
 create mode 100644 tests/reference/ast_new-boolOp1-478328f.json
 create mode 100644 tests/reference/ast_new-boolOp1-478328f.stdout
 create mode 100644 tests/reference/ast_new-class_def1-fe69291.json
 create mode 100644 tests/reference/ast_new-class_def1-fe69291.stdout
 create mode 100644 tests/reference/ast_new-class_def2-c6db986.json
 create mode 100644 tests/reference/ast_new-class_def2-c6db986.stdout
 create mode 100644 tests/reference/ast_new-comment2-f0984d5.json
 create mode 100644 tests/reference/ast_new-comment2-f0984d5.stdout
 create mode 100644 tests/reference/ast_new-comprehension1-69cf2af.json
 create mode 100644 tests/reference/ast_new-comprehension1-69cf2af.stdout
 create mode 100644 tests/reference/ast_new-conditional_expr1-07ccb9e.json
 create mode 100644 tests/reference/ast_new-conditional_expr1-07ccb9e.stdout
 create mode 100644 tests/reference/ast_new-dictionary1-445e718.json
 create mode 100644 tests/reference/ast_new-dictionary1-445e718.stdout
 create mode 100644 tests/reference/ast_new-ellipsis2-3a9750b.json
 create mode 100644 tests/reference/ast_new-ellipsis2-3a9750b.stdout
 create mode 100644 tests/reference/ast_new-for1-887432e.json
 create mode 100644 tests/reference/ast_new-for1-887432e.stdout
 create mode 100644 tests/reference/ast_new-for2-af08901.json
 create mode 100644 tests/reference/ast_new-for2-af08901.stdout
 create mode 100644 tests/reference/ast_new-function_def1-1a872df.json
 create mode 100644 tests/reference/ast_new-function_def1-1a872df.stdout
 create mode 100644 tests/reference/ast_new-function_def2-52c4587.json
 create mode 100644 tests/reference/ast_new-function_def2-52c4587.stdout
 create mode 100644 tests/reference/ast_new-function_def3-f66064a.json
 create mode 100644 tests/reference/ast_new-function_def3-f66064a.stdout
 create mode 100644 tests/reference/ast_new-global1-38edfbd.json
 create mode 100644 tests/reference/ast_new-global1-38edfbd.stdout
 create mode 100644 tests/reference/ast_new-if1-db43586.json
 create mode 100644 tests/reference/ast_new-if1-db43586.stdout
 create mode 100644 tests/reference/ast_new-if2-c3b6022.json
 create mode 100644 tests/reference/ast_new-if2-c3b6022.stdout
 create mode 100644 tests/reference/ast_new-import1-f643fd3.json
 create mode 100644 tests/reference/ast_new-import1-f643fd3.stdout
 create mode 100644 tests/reference/ast_new-lambda1-260d046.json
 create mode 100644 tests/reference/ast_new-lambda1-260d046.stdout
 create mode 100644 tests/reference/ast_new-lambda2-d84336e.json
 create mode 100644 tests/reference/ast_new-lambda2-d84336e.stdout
 create mode 100644 tests/reference/ast_new-match_stmt1-9e84d24.json
 create mode 100644 tests/reference/ast_new-match_stmt1-9e84d24.stdout
 create mode 100644 tests/reference/ast_new-slice1-9c440e3.json
 create mode 100644 tests/reference/ast_new-slice1-9c440e3.stdout
 create mode 100644 tests/reference/ast_new-statements1-e081093.json
 create mode 100644 tests/reference/ast_new-statements1-e081093.stdout
 create mode 100644 tests/reference/ast_new-statements2-c4cdc5f.json
 create mode 100644 tests/reference/ast_new-statements2-c4cdc5f.stdout
 create mode 100644 tests/reference/ast_new-string1-96b90b3.json
 create mode 100644 tests/reference/ast_new-string1-96b90b3.stdout
 create mode 100644 tests/reference/ast_new-string2-44323ea.json
 create mode 100644 tests/reference/ast_new-string2-44323ea.stdout
 create mode 100644 tests/reference/ast_new-string3-37f35a0.json
 create mode 100644 tests/reference/ast_new-string3-37f35a0.stdout
 create mode 100644 tests/reference/ast_new-try1-a9a22cf.json
 create mode 100644 tests/reference/ast_new-try1-a9a22cf.stdout
 create mode 100644 tests/reference/ast_new-tuple1-29c08af.json
 create mode 100644 tests/reference/ast_new-tuple1-29c08af.stdout
 create mode 100644 tests/reference/ast_new-type_comment1-710ea6c.json
 create mode 100644 tests/reference/ast_new-type_comment1-710ea6c.stdout
 create mode 100644 tests/reference/ast_new-unicode-d3199dc.json
 create mode 100644 tests/reference/ast_new-unicode-d3199dc.stdout
 create mode 100644 tests/reference/ast_new-while1-a4c6382.json
 create mode 100644 tests/reference/ast_new-while1-a4c6382.stdout
 create mode 100644 tests/reference/ast_new-with1-6c88c0f.json
 create mode 100644 tests/reference/ast_new-with1-6c88c0f.stdout
 create mode 100644 tests/reference/ast_new-yield-4c41668.json
 create mode 100644 tests/reference/ast_new-yield-4c41668.stdout
 create mode 100644 tests/reference/c-c_interop1-e215531.json
 create mode 100644 tests/reference/c-c_interop1-e215531.stdout
 create mode 100644 tests/reference/c-expr7-bb2692a.json
 create mode 100644 tests/reference/c-expr7-bb2692a.stderr
 create mode 100644 tests/reference/c-expr7-bb2692a.stdout
 create mode 100644 tests/reference/c-expr_01-28f449f.json
 create mode 100644 tests/reference/c-expr_01-28f449f.stdout
 create mode 100644 tests/reference/c-expr_11-c452314.json
 create mode 100644 tests/reference/c-expr_11-c452314.stdout
 create mode 100644 tests/reference/c-expr_12-93c7780.json
 create mode 100644 tests/reference/c-expr_12-93c7780.stdout
 create mode 100644 tests/reference/c-func_static_01-fc146ec.json
 create mode 100644 tests/reference/c-func_static_01-fc146ec.stdout
 create mode 100644 tests/reference/c-import_order_01-3ebf3c3.json
 create mode 100644 tests/reference/c-import_order_01-3ebf3c3.stdout
 create mode 100644 tests/reference/c-loop1-3e341c7.json
 create mode 100644 tests/reference/c-loop1-3e341c7.stdout
 create mode 100644 tests/reference/c-loop4-eec10d3.json
 create mode 100644 tests/reference/c-loop4-eec10d3.stdout
 create mode 100644 tests/reference/c-print_01-4d44628.json
 create mode 100644 tests/reference/c-print_01-4d44628.stdout
 create mode 100644 tests/reference/c-test_import_02-d2c54c4.json
 create mode 100644 tests/reference/c-test_import_02-d2c54c4.stdout
 create mode 100644 tests/reference/c-test_issue_518-fbbd299.json
 create mode 100644 tests/reference/c-test_issue_518-fbbd299.stdout
 create mode 100644 tests/reference/c-variable_decl_03-fa1823b.json
 create mode 100644 tests/reference/c-variable_decl_03-fa1823b.stdout
 create mode 100644 tests/reference/cpp-assert1-ba60925.json
 create mode 100644 tests/reference/cpp-assert1-ba60925.stdout
 create mode 100644 tests/reference/cpp-doconcurrentloop_01-4e9f274.json
 create mode 100644 tests/reference/cpp-doconcurrentloop_01-4e9f274.stdout
 create mode 100644 tests/reference/cpp-expr12-fd2ea87.json
 create mode 100644 tests/reference/cpp-expr12-fd2ea87.stdout
 create mode 100644 tests/reference/cpp-expr15-1661c0d.json
 create mode 100644 tests/reference/cpp-expr15-1661c0d.stdout
 create mode 100644 tests/reference/cpp-expr2-09c05ad.json
 create mode 100644 tests/reference/cpp-expr2-09c05ad.stdout
 create mode 100644 tests/reference/cpp-expr5-1de0e30.json
 create mode 100644 tests/reference/cpp-expr5-1de0e30.stdout
 create mode 100644 tests/reference/cpp-expr6-f337f4f.json
 create mode 100644 tests/reference/cpp-expr6-f337f4f.stdout
 create mode 100644 tests/reference/cpp-expr7-529bd53.json
 create mode 100644 tests/reference/cpp-expr7-529bd53.stderr
 create mode 100644 tests/reference/cpp-expr7-529bd53.stdout
 create mode 100644 tests/reference/cpp-expr8-704cece.json
 create mode 100644 tests/reference/cpp-expr8-704cece.stdout
 create mode 100644 tests/reference/cpp-expr9-48868e9.json
 create mode 100644 tests/reference/cpp-expr9-48868e9.stdout
 create mode 100644 tests/reference/cpp-expr_11-422c839.json
 create mode 100644 tests/reference/cpp-expr_11-422c839.stdout
 create mode 100644 tests/reference/cpp-loop1-0a8cf3b.json
 create mode 100644 tests/reference/cpp-loop1-0a8cf3b.stdout
 create mode 100644 tests/reference/cpp-loop3-6020091.json
 create mode 100644 tests/reference/cpp-loop3-6020091.stdout
 create mode 100644 tests/reference/cpp-loop4-cdb2174.json
 create mode 100644 tests/reference/cpp-loop4-cdb2174.stdout
 create mode 100644 tests/reference/cpp-print_01-026ef17.json
 create mode 100644 tests/reference/cpp-print_01-026ef17.stdout
 create mode 100644 tests/reference/cpp-test_builtin_pow-56b3f92.json
 create mode 100644 tests/reference/cpp-test_builtin_pow-56b3f92.stderr
 create mode 100644 tests/reference/cpp-test_builtin_pow-56b3f92.stdout
 create mode 100644 tests/reference/cpp-test_list_repeat2-698d7f4.json
 create mode 100644 tests/reference/cpp-test_list_repeat2-698d7f4.stdout
 create mode 100644 tests/reference/cpp-test_unary_op_03-fd9669a.json
 create mode 100644 tests/reference/cpp-test_unary_op_03-fd9669a.stdout
 create mode 100644 tests/reference/llvm-assert1-8df4f31.json
 create mode 100644 tests/reference/llvm-assert1-8df4f31.stdout
 create mode 100644 tests/reference/llvm-bindc_01-c984f09.json
 create mode 100644 tests/reference/llvm-bindc_01-c984f09.stdout
 create mode 100644 tests/reference/llvm-bool1-af4376b.json
 create mode 100644 tests/reference/llvm-bool1-af4376b.stdout
 create mode 100644 tests/reference/llvm-expr14-b96b5b1.json
 create mode 100644 tests/reference/llvm-expr14-b96b5b1.stdout
 create mode 100644 tests/reference/llvm-expr_01-54467c1.json
 create mode 100644 tests/reference/llvm-expr_01-54467c1.stdout
 create mode 100644 tests/reference/llvm-func_inline_01-2d4583a.json
 create mode 100644 tests/reference/llvm-func_inline_01-2d4583a.stdout
 create mode 100644 tests/reference/llvm-lpython1-23c5987.json
 create mode 100644 tests/reference/llvm-lpython1-23c5987.stdout
 create mode 100644 tests/reference/llvm-print_04-443a8d8.json
 create mode 100644 tests/reference/llvm-print_04-443a8d8.stdout
 create mode 100644 tests/reference/llvm-structs_11-09fea6a.json
 create mode 100644 tests/reference/llvm-structs_11-09fea6a.stdout
 create mode 100644 tests/reference/llvm-test_issue_518-cdb641a.json
 create mode 100644 tests/reference/llvm-test_issue_518-cdb641a.stdout
 create mode 100644 tests/reference/llvm-test_unary_op_03-046fb86.json
 create mode 100644 tests/reference/llvm-test_unary_op_03-046fb86.stdout
 create mode 100644 tests/reference/llvm_dbg-expr_01-9fc5f30.json
 create mode 100644 tests/reference/llvm_dbg-expr_01-9fc5f30.stdout
 create mode 100644 tests/reference/pass_class_constructor-structs_16-5e3508f.json
 create mode 100644 tests/reference/pass_class_constructor-structs_16-5e3508f.stdout
 create mode 100644 tests/reference/pass_inline_function_calls-func_inline_01-8b6a5da.json
 create mode 100644 tests/reference/pass_inline_function_calls-func_inline_01-8b6a5da.stdout
 create mode 100644 tests/reference/pass_loop_vectorise-vec_01-be9985e.json
 create mode 100644 tests/reference/pass_loop_vectorise-vec_01-be9985e.stdout
 create mode 100644 tests/reference/pass_print_list_tuple-print_02-09600eb.json
 create mode 100644 tests/reference/pass_print_list_tuple-print_02-09600eb.stdout
 create mode 100644 tests/reference/pass_print_list_tuple-print_list_tuple_03-195fa9c.json
 create mode 100644 tests/reference/pass_print_list_tuple-print_list_tuple_03-195fa9c.stdout
 create mode 100644 tests/reference/python-assert1-192ca6c.json
 create mode 100644 tests/reference/python-assert1-192ca6c.stdout
 create mode 100644 tests/reference/python-assign1-f87bafa.json
 create mode 100644 tests/reference/python-assign1-f87bafa.stdout
 create mode 100644 tests/reference/python-expr11-e6681c8.json
 create mode 100644 tests/reference/python-expr11-e6681c8.stdout
 create mode 100644 tests/reference/python-expr17-3b84714.json
 create mode 100644 tests/reference/python-expr17-3b84714.stdout
 create mode 100644 tests/reference/python-expr2-6b69018.json
 create mode 100644 tests/reference/python-expr2-6b69018.stdout
 create mode 100644 tests/reference/python-expr4-161a0ec.json
 create mode 100644 tests/reference/python-expr4-161a0ec.stdout
 create mode 100644 tests/reference/python-expr5-dee0e5c.json
 create mode 100644 tests/reference/python-expr5-dee0e5c.stdout
 create mode 100644 tests/reference/python-expr6-1a1d4fb.json
 create mode 100644 tests/reference/python-expr6-1a1d4fb.stdout
 create mode 100644 tests/reference/run_dbg-test_assert_01-2f34744.json
 create mode 100644 tests/reference/run_dbg-test_assert_01-2f34744.stderr
 create mode 100644 tests/reference/run_dbg-test_assert_02-c6de25a.json
 create mode 100644 tests/reference/run_dbg-test_assert_02-c6de25a.stderr
 create mode 100644 tests/reference/run_dbg-test_assert_03-bd7b7dd.json
 create mode 100644 tests/reference/run_dbg-test_assert_03-bd7b7dd.stderr
 create mode 100644 tests/reference/run_dbg-test_quit_01-30889cc.json
 create mode 100644 tests/reference/run_dbg-test_quit_01-30889cc.stderr
 create mode 100644 tests/reference/run_dbg-test_raise_01-dfd86ca.json
 create mode 100644 tests/reference/run_dbg-test_raise_01-dfd86ca.stderr
 create mode 100644 tests/reference/runtime-test_dict14-421fe53.json
 create mode 100644 tests/reference/runtime-test_dict14-421fe53.stderr
 create mode 100644 tests/reference/runtime-test_dict15-6f3af0d.json
 create mode 100644 tests/reference/runtime-test_dict15-6f3af0d.stderr
 create mode 100644 tests/reference/runtime-test_dict16-c5a958d.json
 create mode 100644 tests/reference/runtime-test_dict16-c5a958d.stderr
 create mode 100644 tests/reference/runtime-test_list_01-3ee9b3e.json
 create mode 100644 tests/reference/runtime-test_list_01-3ee9b3e.stderr
 create mode 100644 tests/reference/runtime-test_list_02-5f7db5f.json
 create mode 100644 tests/reference/runtime-test_list_02-5f7db5f.stderr
 create mode 100644 tests/reference/runtime-test_list_index-0483808.json
 create mode 100644 tests/reference/runtime-test_list_index-0483808.stderr
 create mode 100644 tests/reference/runtime-test_list_index2-1249233.json
 create mode 100644 tests/reference/runtime-test_list_index2-1249233.stderr
 create mode 100644 tests/reference/runtime-test_list_index3-647a94a.json
 create mode 100644 tests/reference/runtime-test_list_index3-647a94a.stderr
 create mode 100644 tests/reference/runtime-test_list_index4-c31dfdb.json
 create mode 100644 tests/reference/runtime-test_list_index4-c31dfdb.stderr
 create mode 100644 tests/reference/runtime-test_str_01-50bdf2f.json
 create mode 100644 tests/reference/runtime-test_str_01-50bdf2f.stdout
 create mode 100644 tests/reference/runtime-test_str_02-c38ba27.json
 create mode 100644 tests/reference/runtime-test_str_02-c38ba27.stdout
 create mode 100644 tests/reference/tokens-comment1-2f8ab90.json
 create mode 100644 tests/reference/tokens-comment1-2f8ab90.stdout
 create mode 100644 tests/reference/tokens-comment2-b289dad.json
 create mode 100644 tests/reference/tokens-comment2-b289dad.stdout
 create mode 100644 tests/reference/tokens-docstring1-1355fbb.json
 create mode 100644 tests/reference/tokens-docstring1-1355fbb.stdout
 create mode 100644 tests/reference/tokens-indent1-0b243c5.json
 create mode 100644 tests/reference/tokens-indent1-0b243c5.stderr
 create mode 100644 tests/reference/tokens-indent1-290e858.json
 create mode 100644 tests/reference/tokens-indent1-290e858.stdout
 create mode 100644 tests/reference/tokens-indent2-a8438a0.json
 create mode 100644 tests/reference/tokens-indent2-a8438a0.stderr
 create mode 100644 tests/reference/tokens-indent2-e702789.json
 create mode 100644 tests/reference/tokens-indent2-e702789.stdout
 create mode 100644 tests/reference/tokens-indent3-0eaf3e6.json
 create mode 100644 tests/reference/tokens-indent3-0eaf3e6.stderr
 create mode 100644 tests/reference/tokens-numbers1-589063f.json
 create mode 100644 tests/reference/tokens-numbers1-589063f.stdout
 create mode 100644 tests/reference/tokens-paren1-94fa736.json
 create mode 100644 tests/reference/tokens-paren1-94fa736.stderr
 create mode 100644 tests/reference/tokens-paren2-0eef339.json
 create mode 100644 tests/reference/tokens-paren2-0eef339.stderr
 create mode 100644 tests/reference/tokens-paren3-0d47100.json
 create mode 100644 tests/reference/tokens-paren3-0d47100.stderr
 create mode 100644 tests/reference/tokens-paren4-5c2fc32.json
 create mode 100644 tests/reference/tokens-paren4-5c2fc32.stderr
 create mode 100644 tests/reference/tokens-symbols1-658c990.json
 create mode 100644 tests/reference/tokens-symbols1-658c990.stdout
 create mode 100644 tests/reference/tokens-test_literal-e20c024.json
 create mode 100644 tests/reference/tokens-test_literal-e20c024.stderr
 create mode 100644 tests/reference/wat-bool1-234bcd1.json
 create mode 100644 tests/reference/wat-bool1-234bcd1.stdout
 create mode 100644 tests/reference/wat-expr14-5e0cb96.json
 create mode 100644 tests/reference/wat-expr14-5e0cb96.stdout
 create mode 100644 tests/reference/wat-expr2-8b17723.json
 create mode 100644 tests/reference/wat-expr2-8b17723.stdout
 create mode 100644 tests/reference/wat-expr9-f73afd1.json
 create mode 100644 tests/reference/wat-expr9-f73afd1.stdout
 create mode 100644 tests/reference/wat-loop1-e0046d4.json
 create mode 100644 tests/reference/wat-loop1-e0046d4.stdout
 create mode 100644 tests/reference/wat-print_str-385e953.json
 create mode 100644 tests/reference/wat-print_str-385e953.stdout
```

Reply:
Upon rerunning these commands, I noticed there are no net changes. All the deleted files reappear.

Reply:
After that, push the changes here.

Reply:
>What is supposed to happen?

That file might get deleted, I think.

Reply:
Did you follow the exact steps in https://github.com/lcompilers/lpython/pull/2587#discussion_r1518922464?

The output shared in https://github.com/lcompilers/lpython/pull/2587#discussion_r1518924090 seems strange and does not look like exact steps were followed.

Reply:
>Upon rerunning these commands, I noticed there are no net changes. All the deleted files reappear.

Can you rebase over the latest main? I think that might help remove the stray file.

Reply:
I believe this has been done from your side. Thanks.

Reply:
I think the file is now removed. We had to rebase over latest main, rebuild the project and then follow the steps in https://github.com/lcompilers/lpython/pull/2587#discussion_r1518922464.

Reply:
Ok, thanks.